
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Section_4_2/">
      
      
        <link rel="next" href="../Section_4_4/">
      
      
      <link rel="icon" href="../assets/Coloss-logo.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.7">
    
    
      
        <title>4.3. SNP and indel detection - wikiCOLOSS: Standard Methods for Apis Omics Research</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.8608ea7d.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Merriweather+Sans:300,300i,400,400i,700,700i%7CRed+Hat+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Merriweather Sans";--md-code-font:"Red Hat Mono"}</style>
      
    
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="slate" data-md-color-primary="yellow" data-md-color-accent="teal">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#4-whole-genome-population-and-association-studies" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="wikiCOLOSS: Standard Methods for Apis Omics Research" class="md-header__button md-logo" aria-label="wikiCOLOSS: Standard Methods for Apis Omics Research" data-md-component="logo">
      
  <img src="../assets/Coloss-logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            wikiCOLOSS: Standard Methods for Apis Omics Research
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              4.3. SNP and indel detection
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="yellow" data-md-color-accent="teal"  aria-label="Dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 7a5 5 0 0 1 5 5 5 5 0 0 1-5 5 5 5 0 0 1-5-5 5 5 0 0 1 5-5m0 2a3 3 0 0 0-3 3 3 3 0 0 0 3 3 3 3 0 0 0 3-3 3 3 0 0 0-3-3m0-7 2.39 3.42C13.65 5.15 12.84 5 12 5s-1.65.15-2.39.42zM3.34 7l4.16-.35A7.2 7.2 0 0 0 5.94 8.5c-.44.74-.69 1.5-.83 2.29zm.02 10 1.76-3.77a7.131 7.131 0 0 0 2.38 4.14zM20.65 7l-1.77 3.79a7.02 7.02 0 0 0-2.38-4.15zm-.01 10-4.14.36c.59-.51 1.12-1.14 1.54-1.86.42-.73.69-1.5.83-2.29zM12 22l-2.41-3.44c.74.27 1.55.44 2.41.44.82 0 1.63-.17 2.37-.44z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="amber" data-md-color-accent="blue"  aria-label="Light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="m17.75 4.09-2.53 1.94.91 3.06-2.63-1.81-2.63 1.81.91-3.06-2.53-1.94L12.44 4l1.06-3 1.06 3zm3.5 6.91-1.64 1.25.59 1.98-1.7-1.17-1.7 1.17.59-1.98L15.75 11l2.06-.05L18.5 9l.69 1.95zm-2.28 4.95c.83-.08 1.72 1.1 1.19 1.85-.32.45-.66.87-1.08 1.27C15.17 23 8.84 23 4.94 19.07c-3.91-3.9-3.91-10.24 0-14.14.4-.4.82-.76 1.27-1.08.75-.53 1.93.36 1.85 1.19-.27 2.86.69 5.83 2.89 8.02a9.96 9.96 0 0 0 8.02 2.89m-1.64 2.02a12.08 12.08 0 0 1-7.8-3.47c-2.17-2.19-3.33-5-3.49-7.82-2.81 3.14-2.7 7.96.31 10.98 3.02 3.01 7.84 3.12 10.98.31"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="wikiCOLOSS: Standard Methods for Apis Omics Research" class="md-nav__button md-logo" aria-label="wikiCOLOSS: Standard Methods for Apis Omics Research" data-md-component="logo">
      
  <img src="../assets/Coloss-logo.png" alt="logo">

    </a>
    wikiCOLOSS: Standard Methods for Apis Omics Research
  </label>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    About
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            About
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../about/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Why a wikiCOLOSS?
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../theteam/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Meet the team
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../chapt_layout/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Chapter Layout
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    1. The omics revolution in Apis: more data than meets the eye
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    2. Sample management
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    3. Genome sequencing
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            3. Genome sequencing
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_3_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    3.1. Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_3_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    3.2. Genome sequencing technologies
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_3_3/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    3.3. The reference genome
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_3_4/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    3.4. Small and large variant detection
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_3_5/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    3.5. Sequencing museum specimens
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6" checked>
        
          
          <label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    4. Whole-genome population and association studies
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_6_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_6">
            <span class="md-nav__icon md-icon"></span>
            4. Whole-genome population and association studies
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.1. Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.2. Ploidy and sampling considerations
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    4.3. SNP and indel detection
    
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    4.3. SNP and indel detection
    
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#43-snp-and-indel-detection" class="md-nav__link">
    <span class="md-ellipsis">
      4.3. SNP and indel detection
    </span>
  </a>
  
    <nav class="md-nav" aria-label="4.3. SNP and indel detection">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#431-mapping-reads-with-bwa-mem-from-fastq-files-to-bam-files" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.1. Mapping reads with BWA-MEM: From FASTQ files to BAM files
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#432-marking-duplicate-reads-with-picard" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.2. Marking duplicate reads with Picard
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#433-base-quality-score-recalibration-bqsr-with-gatk" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.3. Base quality score recalibration (BQSR) with GATK
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#434-calling-variants-with-gatk" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.4. Calling variants with GATK
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#435-combining-all-samples-and-genotypes-with-gatk" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.5. Combining all samples and genotypes with GATK
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#436-filtering-variants-with-gatk-technical-filters" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.6. Filtering variants with GATK: Technical filters
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#437-filtering-variants-with-vcftools-data-quality" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.7. Filtering variants with VCFtools: Data quality
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#438-genotype-phasing" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.8. Genotype phasing
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#439-snp-annotation-with-snpeff" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.9. SNP annotation with SnpEff
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#4310-snp-analysis-by-sequencing-pooled-samples" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.10. SNP analysis by sequencing pooled samples
    </span>
  </a>
  
    <nav class="md-nav" aria-label="4.3.10. SNP analysis by sequencing pooled samples">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#43101-from-fastq-files-to-bam-files" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.10.1. From FASTQ files to BAM files
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#43102-snp-selection-and-pileup-files" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.10.2. SNP selection and pileup files
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#43103-counting-reads-per-allele-with-popoolation" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.10.3. Counting reads per allele with PoPoolation
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_4/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.4. Comparing whole genomes
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_5/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.5. Genome-wide association studies
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_6/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.6. Population genomics: Experimental design
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_7/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.7. Population genomics: Filtering and summary statistics using PLINK
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_8/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.8. Population genomics: Inferring population structure using ADMIXTURE
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_9/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.9. Landscape genomics: An example using LFMM
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_10/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.10. Applying population genomics to conservation: Reduced SNP analysis
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_7" >
        
          
          <label class="md-nav__link" for="__nav_7" id="__nav_7_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    5. Epigenomics
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_7_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_7">
            <span class="md-nav__icon md-icon"></span>
            5. Epigenomics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_5_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    5.1. Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_5_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    5.2. DNA methylation
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_5_3/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    5.3. Epitranscriptomics: RNA methylation of m6A
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_5_4/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    5.4. Chromatin organization and histone modifications
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_5_5/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    5.5. Applications and limitations
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_8" >
        
          
          <label class="md-nav__link" for="__nav_8" id="__nav_8_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    6. Transcriptomics
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_8_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_8">
            <span class="md-nav__icon md-icon"></span>
            6. Transcriptomics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_6_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    6.1. Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_6_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    6.2. Sequencing technologies
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_6_3/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    6.3. Single-cell transcriptomics
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_6_4/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    6.4. Data handling and analysis
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_6_5/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    6.5. Applications and limitations
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_9" >
        
          
          <label class="md-nav__link" for="__nav_9" id="__nav_9_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    7. Functional genomics and xenobiotic treatment
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_9_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_9">
            <span class="md-nav__icon md-icon"></span>
            7. Functional genomics and xenobiotic treatment
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_7_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    7.1. Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_7_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    7.2. CRISPR
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_7_3/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    7.3. RNA interference
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_7_4/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    7.4. Xenobiotic treatment
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_7_5/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    7.5. Applications and limitations
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_10" >
        
          
          <label class="md-nav__link" for="__nav_10" id="__nav_10_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    8. Proteomics
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_10_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_10">
            <span class="md-nav__icon md-icon"></span>
            8. Proteomics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_8_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    8.1. Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_8_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    8.2. Standard methods for shot-gun proteomics sample preparation
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_8_3/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    8.3. Liquid chromatography and mass spectrometry
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_8_4/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    8.4. Proteomics data processing
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_8_5/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    8.5. Applications and limitations
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_11" >
        
          
          <label class="md-nav__link" for="__nav_11" id="__nav_11_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    9. Metabolomics
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_11_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_11">
            <span class="md-nav__icon md-icon"></span>
            9. Metabolomics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_9_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    9.1. Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_9_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    9.2. Sample preparation for metabolomics
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_9_3/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    9.3. Chromatography and mass spectrometry
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_9_4/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    9.4. Metabolomics data processing
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_9_5/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    9.5. Metabolomics applications and limitations
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_12" >
        
          
          <label class="md-nav__link" for="__nav_12" id="__nav_12_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    10. Microbiome analysis
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_12_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_12">
            <span class="md-nav__icon md-icon"></span>
            10. Microbiome analysis
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_10_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    10.1. Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_10_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    10.2. Sampling and DNA extraction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_10_3/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    10.3. Amplicon sequencing
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_10_4/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    10.4. Microbiome data analysis
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_10_5/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    10.5. Applications and limitations
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_13" >
        
          
          <label class="md-nav__link" for="__nav_13" id="__nav_13_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    11. Data management and open access sharing
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_13_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_13">
            <span class="md-nav__icon md-icon"></span>
            11. Data management and open access sharing
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_11_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    11.1. Metadata standardization
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_11_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    11.2. Sharing pipelines and scripts
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_12/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    12. The future of Apis omics: Biological integration
    
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#43-snp-and-indel-detection" class="md-nav__link">
    <span class="md-ellipsis">
      4.3. SNP and indel detection
    </span>
  </a>
  
    <nav class="md-nav" aria-label="4.3. SNP and indel detection">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#431-mapping-reads-with-bwa-mem-from-fastq-files-to-bam-files" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.1. Mapping reads with BWA-MEM: From FASTQ files to BAM files
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#432-marking-duplicate-reads-with-picard" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.2. Marking duplicate reads with Picard
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#433-base-quality-score-recalibration-bqsr-with-gatk" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.3. Base quality score recalibration (BQSR) with GATK
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#434-calling-variants-with-gatk" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.4. Calling variants with GATK
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#435-combining-all-samples-and-genotypes-with-gatk" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.5. Combining all samples and genotypes with GATK
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#436-filtering-variants-with-gatk-technical-filters" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.6. Filtering variants with GATK: Technical filters
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#437-filtering-variants-with-vcftools-data-quality" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.7. Filtering variants with VCFtools: Data quality
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#438-genotype-phasing" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.8. Genotype phasing
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#439-snp-annotation-with-snpeff" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.9. SNP annotation with SnpEff
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#4310-snp-analysis-by-sequencing-pooled-samples" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.10. SNP analysis by sequencing pooled samples
    </span>
  </a>
  
    <nav class="md-nav" aria-label="4.3.10. SNP analysis by sequencing pooled samples">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#43101-from-fastq-files-to-bam-files" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.10.1. From FASTQ files to BAM files
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#43102-snp-selection-and-pileup-files" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.10.2. SNP selection and pileup files
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#43103-counting-reads-per-allele-with-popoolation" class="md-nav__link">
    <span class="md-ellipsis">
      4.3.10.3. Counting reads per allele with PoPoolation
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  



<h1 id="4-whole-genome-population-and-association-studies">4. Whole-genome population and association studies</h1>
<h2 id="43-snp-and-indel-detection">4.3. SNP and indel detection</h2>
<p>Here we provide a primer on the bioinformatics involved in detecting sequence variants from <em>Illumina</em> data, as it is often very difficult to know how to start. The software tools chosen here result from the authors’ positive experience in their application of <em>A. mellifera</em> genomics workflow but do not reject the validity and rapidity of alternative algorithms and pipelines. We recommend the review by Bourgeois and Warren (2021) listing the most common software and methods applicable to comparative and population genomics in eukaryotes. For further details on each parameter, future users will have to go to the dedicated websites for <strong>BWA-MEM</strong>, <strong>SAMtools</strong>, <strong>GATK</strong>, <strong>Picard, VCFtools,</strong> and <strong>BCFtools</strong> (Danecek et al., 2011, 2021; McKenna et al., 2010).</p>
<h3 id="431-mapping-reads-with-bwa-mem-from-fastq-files-to-bam-files">4.3.1. Mapping reads with <strong>BWA-MEM</strong>: From FASTQ files to BAM files</h3>
<p>The first step is to map the short reads to the reference genome to produce a BAM file. Typically, for SNP detection, sequencing is performed paired-end with an <em>Illumina</em> instrument, giving millions of \~150 bp reads in which 300-500 bp will separate the beginning of Read 1 (first mate or forward read) and the end of Read 2 (second mate or reverse read), according to the size specification of the sequencing library. For each sample, the sequencing instrument will produce two files in compressed FASTQ format containing the reads. These will have names in the form “AOC5_TAGCTT_L002_R1.fastq.gz” and “AOC5_TAGCTT_L002_R2.fastq.gz”, which includes the sample name (AOC5), the sample identification in the sample sheet (TAGCTT), the lane number on the sequencing machine (L002) and the Read (R1 or R2). One file will contain all Read 1 sequences and another will contain all Read 2 sequences.</p>
<p>In honey bee genomics, there is currently no consensus on a single best mapping software. However, based on our literature survey, we found that <strong>BWA-MEM</strong> (Burrows–Wheeler Aligner), <strong>Bowtie2</strong> and <strong>Stampy</strong> have been used for <em>A. mellifera</em> and <em>A. cerana</em> short DNA read mapping. However, we noted that <strong>BWA-MEM</strong> was the most commonly used, and it was also reported as one of the top performing short reads aligners (Musich et al., 2021). Therefore, we have chosen BWA-MEM as our current standard software for mapping honey bee genomes.</p>
<p><strong>Step 1.</strong> Mapping the reads to the reference genome is done here with <strong>BWA-MEM</strong> (Li, 2013) and parsing with <strong>SAMtools</strong> using the following commands:</p>
<pre><code class="language-bash">bwa mem -M GCF_003254395.2_Amel_HAv3.1_genomic.fna \
AOC5_TAGCTT_L002_R1.fastq.gz \
AOC5_TAGCTT_L002_R2.fastq.gz | \
samtools view -bh -o AOC5_TAGCTT_L002_aligned.bam -
</code></pre>
<p>The -M command marks shorter split reads as secondary. As <strong>BWA-MEM</strong> will output a very large file in SAM format, its output is piped (|) directly into the samtools view -bh command, that will compress directly in the compact BAM format (-b), while including the header (-h).</p>
<p>The full description of the SAM format can be found <a href="https://samtools.github.io/hts-specs/SAMv1.pdf">online</a>, and is updated on a regular basis. Briefly, a SAM/BAM file will contain a header with diverse information, such as if the file was sorted, the name of the reference sequence used, and the programs that were used to process the data (alignment and other subsequent analyses).</p>
<p><strong>Special considerations</strong>: If a sample was sequenced in two separate runs, lanes or from two different libraries, the best practice is to require the mapping to be done separately and the BAM files produced to be subsequently merged. This is important for backwards traceability and for downstream analyses, as reads from different runs may have specific biases affecting base or mapping quality estimations. In this scenario, sample “BER15” was sequenced in two separate lanes of an <em>Illumina</em> instrument. Alignment was done separately, giving two BAM files, to be merged with samtools merge.</p>
<p><strong>Step 2.</strong> Merge BAM files with mapped reads using <strong>SAMtools</strong>:</p>
<pre><code class="language-bash">samtools merge -o BER15_ATCACG_merged_aligned.bam \
BER15_ATCACG_L002_aligned.bam \
BER15_ATCACG_L005_aligned.bam
</code></pre>
<p>This information is passed on to the BAM file in the mandatory ID tag of the read group (\@RG) lines in the header, and each mate-pair read in the BAM file will be assigned to one or the other ID tag:</p>
<pre><code class="language-bash">samtools view -H BER15_ATCACG_merged_aligned.bam | grep ^@RG
@RG     ID:BER15_ATCACG_L002
@RG     ID:BER15_ATCACG_L005
</code></pre>
<p><strong>Step 3.</strong> Sort the BAM file using the following commands with <strong>Picard</strong>:</p>
<pre><code class="language-bash">java -jar picard.jar SortSam \
    I=AOC5_TAGCTT_L002_aligned.bam \
    O=AOC5_TAGCTT_L002_sorted.bam \
    SORT_ORDER=coordinate
</code></pre>
<h3 id="432-marking-duplicate-reads-with-picard">4.3.2. Marking duplicate reads with <strong>Picard</strong></h3>
<p>During sample preparation and sequencing, DNA amplification steps will be performed, in which case a single DNA fragment can produce duplicate reads. It is important to tag such reads in the BAM file, so that they will be ignored when calling variants with <strong>GATK</strong> HaplotypeCaller.</p>
<p><strong>Step 4.</strong> Tag the duplicate reads using <strong>Picard</strong> option MarkDuplicates:</p>
<pre><code class="language-bash">java -jar picard.jar MarkDuplicates \
    I=AOC5_TAGCTT_L002_sorted.bam \
    O=AOC5_TAGCTT_L002_dedup.bam \
    M=marked_dup_metrics.txt
</code></pre>
<p><strong>Step 5.</strong> The reads can then be sorted and indexed with <strong>SAMtools</strong>:</p>
<pre><code class="language-bash">samtools sort -T AOC5_TAGCTT_L002_dedup.bam \
-o AOC5_TAGCTT_L002_sort.bam
samtools index AOC5_TAGCTT_L002_sort.bam
</code></pre>
<h3 id="433-base-quality-score-recalibration-bqsr-with-gatk">4.3.3. Base quality score recalibration (BQSR) with <strong>GATK</strong></h3>
<p>Sequence reads in the FASTQ format have a quality score associated to each base called by the sequencing machine, which are based on the manufacturer’s algorithms. These express the confidence of the base calling and will greatly influence the algorithms used for variant calling as well as deciding between sequencing errors and real biological differences.</p>
<p>BQSR models non-random technical errors in the data using a machine learning approach and adjusts the scores accordingly. A set of known variants in the VCF file format is provided to mask bases at sites of expected variation. Mismatches outside these sites are counted as errors.</p>
<p><strong>Step 6.</strong> Use the following <strong>GATK</strong> commands to generate the recalibration table:</p>
<pre><code class="language-bash">gatk BaseRecalibrator \
  -I AOC5_TAGCTT_L002_sort.bam \
  -R GCF_003254395.2_Amel_Hav3.1_genomic.fna \
  --known-sites sites_of_variation.vcf \
  --known-sites another/optional/setOfSitesToMask.vcf \
  -O recal_data.table
</code></pre>
<p><strong>Step 7.</strong> Recalibrate base qualities:</p>
<pre><code class="language-bash">gatk ApplyBQSR \
  -R GCF_003254395.2_Amel_Hav3.1_genomic.fna \
  -I AOC5_TAGCTT_L002_sort.bam \
  --bqsr-recal-file recal_data.table \
  -O AOC5_TAGCTT_L002_BQSR.bam
</code></pre>
<h3 id="434-calling-variants-with-gatk">4.3.4. Calling variants with <strong>GATK</strong></h3>
<p><strong>Step 8.</strong> Variants can be called for each sample individually using the following <strong>GATK</strong> commands:</p>
<pre><code class="language-bash">gatk HaplotypeCaller \
  -R GCF_003254395.2_Amel_Hav3.1_genomic.fna \
  -I AOC5_TAGCTT_L002_BQSR.bam \
  -O AOC5.g.vcf.gz
  -ERC GVCF \
  -ploidy 2
</code></pre>
<p><strong>Special considerations:</strong> The option “-ploidy” is set to 2 (diploid) in the example, which fits for a honey bee worker or queen sample. The option “-ploidy 1” can be used for haploid drones, but in our experience, this leads to false positive SNP detection. A better option is to analyze haploid samples using the diploid model and to filter out SNPs with heterozygote calls in the drones after the genotyping step.</p>
<h3 id="435-combining-all-samples-and-genotypes-with-gatk">4.3.5. Combining all samples and genotypes with <strong>GATK</strong></h3>
<p>At this stage, all individual genotype files will be combined into a single file. If more than hundred samples are processed, this may require a large amount of memory, which can be specified (<code>--java-options "-Xmx10g"</code>).</p>
<p><strong>Step 9.</strong> Combine genotype files:</p>
<pre><code class="language-bash">gatk --java-options &quot;-Xmx10g&quot; CombineGVCFs \
  -R GCF_003254395.2_Amel_Hav3.1_genomic.fna \
  --variant AOC1.g.vcf.gz \
  --variant AOC2.g.vcf.gz \
  --variant AOC3.g.vcf.gz \
  --variant AOC4.g.vcf.gz \
  --variant AOC5.g.vcf.gz \
  --o all_samples.g.vcf.gz
</code></pre>
<p><strong>Step 10.</strong> Once combined in a large file, joint genotyping can be performed:</p>
<pre><code class="language-bash">gatk --java-options &quot;-Xmx10g&quot; GenotypeGVCFs \
  -R GCF_003254395.2_Amel_Hav3.1_genomic.fna \
  -V all_samples.g.vcf.gz \
  --use-new-qual-calculator \
  -O all_samples_genotyped.vcf.gz
</code></pre>
<h3 id="436-filtering-variants-with-gatk-technical-filters">4.3.6. Filtering variants with <strong>GATK</strong>: Technical filters</h3>
<p><strong>Step 11.</strong> The following commands will retain only site variants coded as SNPs:</p>
<pre><code class="language-bash">gatk SelectVariants \
  -R GCF_003254395.2_Amel_Hav3.1_genomic.fna \
  -V all_samples_genotyped.vcf.gz \
  --select-type-to-include SNP \
  -O all_samples_genotyped_SNPs.vcf.gz
</code></pre>
<p><strong>Step 12.</strong> Variants can then be filtered based on quality scores. First, mark the SNPs in the VCF file that do not pass quality score thresholds:</p>
<pre><code class="language-bash">gatk VariantFiltration \
  -R GCF_003254395.2_Amel_Hav3.1_genomic.fna \
  --filter-expression « QD &lt; 2.0 || FS &gt; 60.0 || MQ &lt; 40.0 »
  --filterName “quality_filters”
  -V all_samples_genotyped_SNPs.vcf.gz \
  -O all_samples_genotyped_SNPs_filter_tagged.vcf.gz
</code></pre>
<p>Here, we adopted the recommended and generic hard-filtering parameters of all SNP with a variant confidence (QD) below 2, a strand bias (FS) over 60 and a mapping quality (MQ) below 40. However, we recommend readers to always plot these statistics distribution using <strong>R</strong> and <strong>ggplot2</strong> and verify that the thresholds is adapted to their dataset.</p>
<p><strong>Step 13.</strong> Then, the SNPs that passed the filters can be extracted:</p>
<pre><code class="language-bash">gatk SelectVariants \
  -R GCF_003254395.2_Amel_Hav3.1_genomic.fna \
  --exclude-filtered
  -V all_samples_genotyped_SNPs_filter_tagged.vcf.gz
  -O all_samples_genotyped_SNPs_filter_passed.vcf.gz
</code></pre>
<p><strong>Special considerations:</strong> If variants were detected in haploid drones using the diploid model, markers having heterozygote calls can now be removed.</p>
<h3 id="437-filtering-variants-with-vcftools-data-quality">4.3.7. Filtering variants with <strong>VCFtools</strong>: Data quality</h3>
<p>To ensure a high-quality dataset, SNPs should be filtered using not only quality scores, but also, for instance, the mean depth and missing data. Here, the software <strong>VCFtools</strong> (Danecek et al., 2011) is used to include biallelic SNPs (--max-alleles) with at least 5 read depth (--min-meanDP). Further information about SNP filtering, such as the correlation between markers or linkage disequilibrium (LD), can be found in <a href="https://maevatecher.github.io/standard-methods-apis-omics/Part_4_7/">Section 4.7</a>.</p>
<pre><code class="language-bash">vcftools \
--gzvcf all_samples_genotyped_SNPs_filter_passed.vcf.gz \
--max-alleles 2 \
--min-meanDP 5 \
--out all_samples_filter_2.vcf.gz
</code></pre>
<h3 id="438-genotype-phasing">4.3.8. Genotype phasing</h3>
<p>Phasing refers to the process of statistical estimation of haplotypes from genotype data. While drone sequencing generates directly phased data, worker or queen sequencing requires an extra step in the analysis for phasing the diploid data. There are several freely available programs for this purpose, such as<a href="http://faculty.washington.edu/browning/beagle/beagle.html"><strong>Beagle</strong></a> <strong>5.4</strong> (Browning et al., 2021),<a href="http://stephenslab.uchicago.edu/software.html#fastphase"><strong>fastPHASE</strong></a>(Scheet &amp; Stephens, 2006),<a href="http://mathgen.stats.ox.ac.uk/impute/impute_v2.html"><strong>IMPUTE2</strong></a> (Howie et al., 2009), and <strong>SHAPEIT4</strong> (Delaneau et al., 2019). <strong>SHAPEIT4</strong> is highly accurate, simple, and computationally fast in large datasets (De Marino et al., 2022). The program can be implemented using the following command line:</p>
<pre><code class="language-bash">shapeit4.2 --input all_samples_filter_2.vcf.gz \
--map genomic_map.gmap.gz \
--output phased.vcf.gz
</code></pre>
<h3 id="439-snp-annotation-with-snpeff">4.3.9. SNP annotation with <strong>SnpEff</strong></h3>
<p><strong>SnpEff</strong> is a toolbox to annotate the variants and to calculate their effects (Cingolani et al., 2012). <strong>SnpEff</strong> can be implemented on the <a href="https://usegalaxy.org/;%20see%20section%204.9.2%20step17">Galaxy</a> analysis platform or by using the following command line:</p>
<pre><code class="language-bash">java -Xmx8g -jar snpEff.jar Apis_mellifera all_samples_filter_2.vcf.gz &gt; all_samples_filter_2_annotated.vcf.gz
</code></pre>
<h3 id="4310-snp-analysis-by-sequencing-pooled-samples">4.3.10. SNP analysis by sequencing pooled samples</h3>
<p>Pooling offers an alternative for screening populations or subspecies using genome-wide markers and high-throughput technologies. Group-level data obtained from genotyping experiments will deviate from individual-level data in the sense that frequencies of the different alleles observed in the pool, rather than genotype polymorphisms, will be obtained (see <a href="https://maevatecher.github.io/standard-methods-apis-omics/Part_4_2/">Section 4.2</a>). Allele frequencies can be used directly or transformed into mean genotype using <strong>PLINK</strong> (see <a href="https://maevatecher.github.io/standard-methods-apis-omics/Part_4_5/">Section 4.5</a>), or dedicated genotype reconstruction methods. Such methods are currently available to use such allele frequencies, from drone offspring or from a worker pool (Eynard et al., 2022) to obtain queen genotypes. The procedure described here allows the estimation of allele frequencies in pooled samples by using the <strong>PoPoolation</strong> software (Kofler et al., 2011).</p>
<h4 id="43101-from-fastq-files-to-bam-files">4.3.10.1. From FASTQ files to BAM files</h4>
<p>Mapping reads and duplicate reads detection are performed as described above (see <a href="https://maevatecher.github.io/standard-methods-apis-omics/Part_4_3/">Section 4.3</a>).</p>
<h4 id="43102-snp-selection-and-pileup-files">4.3.10.2. SNP selection and pileup files</h4>
<p>Each line in a PILEUP file describes a single position in the genome. It contains information on the number of reads “piled up” at the position, together with the base called for each read, if the reads mapped to the forward or the reverse strand and quality scores for the base called for each read. PILEUP files are very large and the detection of SNPs in pooled sequencing is not very effective, so it is best to work on a list of high-quality SNPs that were detected in a previous experiment.</p>
<pre><code class="language-bash">samtools mpileup -I -l snp.list \
    -f reference_genome.fa \
    -C 50 -q 20 -Q 20 sample.bam \
    -o sample.pileup
</code></pre>
<p>The option -I is for skipping indels. The list snp.list is in the form of chromosome position (tab separated). This will generate one PILEUP file per sample. A list of BAM files can be provided (option -l), to make a mpileup file containing multiple samples.</p>
<pre><code class="language-bash">samtools mpileup -I -l snp.list \
    -f reference_genome.fa \
    -C 50 -q 20 -Q 20 -b bams.list \
    -o multiple_samples.mpileup
</code></pre>
<h4 id="43103-counting-reads-per-allele-with-popoolation">4.3.10.3. Counting reads per allele with PoPoolation</h4>
<p><strong>PoPoolation</strong> (Kofler et al., 2011) takes PILEUP files as entries and generates read counts for each possible nucleotide base (A, C, G, or T) at each SNP position reported in the PILEUP file.</p>
<pre><code class="language-bash">java -ea -Xmx10g -jar mpileup2sync.jar \
                        --input sample.pileup \
                        --output sample.sync \
                        --fastq-type sanger
</code></pre>
<p>The sample.sync files present the results in the form of one line per SNP position, with a count of reads for each of A:T:C:G:N:del. The <strong>PoPoolation</strong> suite provides some perl scripts to calculate allele frequency differences or F~ST~ (fixation index) values between pairs of pooled DNA sequences.</p>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": [], "search": "../assets/javascripts/workers/search.f8cc74c7.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.c8b220af.min.js"></script>
      
    
  </body>
</html>