
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../theteam/">
      
      
        <link rel="next" href="../Section_1/">
      
      
      <link rel="icon" href="../assets/Coloss-logo.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.7">
    
    
      
        <title>Chapter Layout - wikiCOLOSS: Standard Methods for Apis Omics Research</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.8608ea7d.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Merriweather+Sans:300,300i,400,400i,700,700i%7CRed+Hat+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Merriweather Sans";--md-code-font:"Red Hat Mono"}</style>
      
    
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="slate" data-md-color-primary="yellow" data-md-color-accent="teal">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#1-the-omics-revolution-in-apis-more-data-than-meets-the-eye" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="wikiCOLOSS: Standard Methods for Apis Omics Research" class="md-header__button md-logo" aria-label="wikiCOLOSS: Standard Methods for Apis Omics Research" data-md-component="logo">
      
  <img src="../assets/Coloss-logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            wikiCOLOSS: Standard Methods for Apis Omics Research
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Chapter Layout
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="yellow" data-md-color-accent="teal"  aria-label="Dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 7a5 5 0 0 1 5 5 5 5 0 0 1-5 5 5 5 0 0 1-5-5 5 5 0 0 1 5-5m0 2a3 3 0 0 0-3 3 3 3 0 0 0 3 3 3 3 0 0 0 3-3 3 3 0 0 0-3-3m0-7 2.39 3.42C13.65 5.15 12.84 5 12 5s-1.65.15-2.39.42zM3.34 7l4.16-.35A7.2 7.2 0 0 0 5.94 8.5c-.44.74-.69 1.5-.83 2.29zm.02 10 1.76-3.77a7.131 7.131 0 0 0 2.38 4.14zM20.65 7l-1.77 3.79a7.02 7.02 0 0 0-2.38-4.15zm-.01 10-4.14.36c.59-.51 1.12-1.14 1.54-1.86.42-.73.69-1.5.83-2.29zM12 22l-2.41-3.44c.74.27 1.55.44 2.41.44.82 0 1.63-.17 2.37-.44z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="amber" data-md-color-accent="blue"  aria-label="Light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="m17.75 4.09-2.53 1.94.91 3.06-2.63-1.81-2.63 1.81.91-3.06-2.53-1.94L12.44 4l1.06-3 1.06 3zm3.5 6.91-1.64 1.25.59 1.98-1.7-1.17-1.7 1.17.59-1.98L15.75 11l2.06-.05L18.5 9l.69 1.95zm-2.28 4.95c.83-.08 1.72 1.1 1.19 1.85-.32.45-.66.87-1.08 1.27C15.17 23 8.84 23 4.94 19.07c-3.91-3.9-3.91-10.24 0-14.14.4-.4.82-.76 1.27-1.08.75-.53 1.93.36 1.85 1.19-.27 2.86.69 5.83 2.89 8.02a9.96 9.96 0 0 0 8.02 2.89m-1.64 2.02a12.08 12.08 0 0 1-7.8-3.47c-2.17-2.19-3.33-5-3.49-7.82-2.81 3.14-2.7 7.96.31 10.98 3.02 3.01 7.84 3.12 10.98.31"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="wikiCOLOSS: Standard Methods for Apis Omics Research" class="md-nav__button md-logo" aria-label="wikiCOLOSS: Standard Methods for Apis Omics Research" data-md-component="logo">
      
  <img src="../assets/Coloss-logo.png" alt="logo">

    </a>
    wikiCOLOSS: Standard Methods for Apis Omics Research
  </label>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" checked>
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    About
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            About
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../about/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Why a wikiCOLOSS?
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../theteam/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Meet the team
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Chapter Layout
    
  </span>
  

      </a>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    1. The omics revolution in Apis: more data than meets the eye
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    2. Sample management
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    3. Genome sequencing
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            3. Genome sequencing
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_3_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    3.1. Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_3_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    3.2. Genome sequencing technologies
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_3_3/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    3.3. The reference genome
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_3_4/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    3.4. Small and large variant detection
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_3_5/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    3.5. Sequencing museum specimens
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6" >
        
          
          <label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    4. Whole-genome population and association studies
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_6">
            <span class="md-nav__icon md-icon"></span>
            4. Whole-genome population and association studies
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.1. Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.2. Ploidy and sampling considerations
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_3/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.3. SNP and indel detection
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_4/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.4. Comparing whole genomes
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_5/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.5. Genome-wide association studies
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_6/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.6. Population genomics: Experimental design
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_7/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.7. Population genomics: Filtering and summary statistics using PLINK
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_8/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.8. Population genomics: Inferring population structure using ADMIXTURE
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_9/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.9. Landscape genomics: An example using LFMM
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_10/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.10. Applying population genomics to conservation: Reduced SNP analysis
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_7" >
        
          
          <label class="md-nav__link" for="__nav_7" id="__nav_7_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    5. Epigenomics
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_7_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_7">
            <span class="md-nav__icon md-icon"></span>
            5. Epigenomics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_5_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    5.1. Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_5_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    5.2. DNA methylation
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_5_3/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    5.3. Epitranscriptomics: RNA methylation of m6A
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_5_4/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    5.4. Chromatin organization and histone modifications
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_5_5/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    5.5. Applications and limitations
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_8" >
        
          
          <label class="md-nav__link" for="__nav_8" id="__nav_8_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    6. Transcriptomics
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_8_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_8">
            <span class="md-nav__icon md-icon"></span>
            6. Transcriptomics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_6_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    6.1. Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_6_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    6.2. Sequencing technologies
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_6_3/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    6.3. Single-cell transcriptomics
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_6_4/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    6.4. Data handling and analysis
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_6_5/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    6.5. Applications and limitations
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_9" >
        
          
          <label class="md-nav__link" for="__nav_9" id="__nav_9_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    7. Functional genomics and xenobiotic treatment
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_9_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_9">
            <span class="md-nav__icon md-icon"></span>
            7. Functional genomics and xenobiotic treatment
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_7_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    7.1. Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_7_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    7.2. CRISPR
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_7_3/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    7.3. RNA interference
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_7_4/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    7.4. Xenobiotic treatment
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_7_5/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    7.5. Applications and limitations
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_10" >
        
          
          <label class="md-nav__link" for="__nav_10" id="__nav_10_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    8. Proteomics
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_10_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_10">
            <span class="md-nav__icon md-icon"></span>
            8. Proteomics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_8_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    8.1. Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_8_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    8.2. Standard methods for shot-gun proteomics sample preparation
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_8_3/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    8.3. Liquid chromatography and mass spectrometry
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_8_4/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    8.4. Proteomics data processing
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_8_5/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    8.5. Applications and limitations
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_11" >
        
          
          <label class="md-nav__link" for="__nav_11" id="__nav_11_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    9. Metabolomics
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_11_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_11">
            <span class="md-nav__icon md-icon"></span>
            9. Metabolomics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_9_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    9.1. Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_9_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    9.2. Sample preparation for metabolomics
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_9_3/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    9.3. Chromatography and mass spectrometry
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_9_4/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    9.4. Metabolomics data processing
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_9_5/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    9.5. Metabolomics applications and limitations
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_12" >
        
          
          <label class="md-nav__link" for="__nav_12" id="__nav_12_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    10. Microbiome analysis
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_12_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_12">
            <span class="md-nav__icon md-icon"></span>
            10. Microbiome analysis
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_10_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    10.1. Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_10_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    10.2. Sampling and DNA extraction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_10_3/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    10.3. Amplicon sequencing
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_10_4/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    10.4. Microbiome data analysis
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_10_5/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    10.5. Applications and limitations
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_13" >
        
          
          <label class="md-nav__link" for="__nav_13" id="__nav_13_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    11. Data management and open access sharing
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_13_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_13">
            <span class="md-nav__icon md-icon"></span>
            11. Data management and open access sharing
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_11_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    11.1. Metadata standardization
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_11_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    11.2. Sharing pipelines and scripts
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_12/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    12. The future of Apis omics: Biological integration
    
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  



<h1 id="1-the-omics-revolution-in-apis-more-data-than-meets-the-eye"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_1/">1. The â€˜omics revolution in Apis: More data than meets the eye</a></h1>
<h1 id="2-sample-management"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_2/">2. Sample management</a></h1>
<h1 id="3-genome-sequencing"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_1/">3. Genome sequencing</a></h1>
<h2 id="31-introduction"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_1/">3.1. Introduction</a></h2>
<h2 id="32-genome-sequencing-technologies"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_2/">3.2. Genome sequencing technologies</a></h2>
<h3 id="321-sanger-sequencing"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_2/#321-sanger-sequencing">3.2.1. Sanger sequencing</a></h3>
<h3 id="322-next-generation-sequencing"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_2/#322-next-generation-sequencing">3.2.2. Next generation sequencing</a></h3>
<h3 id="323-long-read-sequencing"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_2/#323-long-read-sequencing">3.2.3. Long-read sequencing</a></h3>
<h2 id="33-the-reference-genome"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_3/">3.3. The reference genome</a></h2>
<h3 id="331-assembling-the-reference-genome"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_3/#331-assembling-the-reference-genome">3.3.1. Assembling the reference genome</a></h3>
<h4 id="3311-hi-c-chromosome-conformation-capture"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_3/#3311-hi-c-chromosome-conformation-capture">3.3.1.1. Hi-C chromosome conformation capture</a></h4>
<h4 id="3312-dna-source-selection"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_3/#3312-dna-source-selection">3.3.1.2. DNA source selection</a></h4>
<h3 id="332-annotating-the-reference-genome"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_3/#332-annotating-the-reference-genome">3.3.2. Annotating the reference genome</a></h3>
<h3 id="333-high-molecular-weight-dna-extraction"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_3/#333-high-molecular-weight-dna-extraction">3.3.3. High molecular weight DNA extraction</a></h3>
<h2 id="34-small-and-large-variant-detection"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_4/">3.4. Small and large variant detection</a></h2>
<h3 id="341-rad-seq"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_4/#341-rad-seq">3.4.1. RAD-seq</a></h3>
<h2 id="35-sequencing-museum-specimens"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_5/">3.5. Sequencing museum specimens</a></h2>
<h3 id="351-considerations"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_5/#351-considerations">3.5.1. Considerations</a></h3>
<h3 id="352-materials"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_5/#352-materials">3.5.2. Materials</a></h3>
<h3 id="353-procedure"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_5/#353-procedure">3.5.3. Procedure</a></h3>
<h4 id="3531-preparation-and-lysis"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_5/#3531-preparation-and-lysis">3.5.3.1. Preparation and lysis</a></h4>
<h4 id="3532-dna-extraction"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_5/#3532-dna-extraction">3.5.3.2. DNA extraction</a></h4>
<h4 id="3533-precipitation"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_5/#3533-precipitation">3.5.3.3. Precipitation</a></h4>
<h4 id="3534-washing"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_5/#3534-washing">3.5.3.4. Washing</a></h4>
<h4 id="3535-final-solubilization"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_5/#3535-final-solubilization">3.5.3.5. Final solubilization</a></h4>
<h3 id="354-sequencing-of-museum-and-ancient-genomes"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_5/#354-sequencing-of-museum-and-ancient-genomes">3.5.4. Sequencing of museum and ancient genomes</a></h3>
<h3 id="355-guidance-on-the-data-analysis-methods"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_5/#355-guidance-on-the-data-analysis-methods">3.5.5. Guidance on the data analysis methods</a></h3>
<h3 id="356-applications-and-limitations"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_3_5/#356-applications-and-limitations">3.5.6. Applications and limitations</a></h3>
<h1 id="4-whole-genome-population-and-association-studies"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_1/">4. Whole-genome population and association studies</a></h1>
<h2 id="41-introduction"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_1/">4.1. Introduction</a></h2>
<h2 id="42-ploidy-and-sampling-considerations"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_2/">4.2. Ploidy and sampling considerations</a></h2>
<h3 id="421-individual-sampling"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_2/#421-individual-sampling">4.2.1. Individual sampling</a></h3>
<h3 id="422-pooled-sampling"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_2/#422-pooled-sampling">4.2.2. Pooled sampling</a></h3>
<h4 id="4221-groups-of-workers"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_2/#4221-groups-of-workers">4.2.2.1. Groups of workers</a></h4>
<h4 id="4222-groups-of-drones"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_2/#4222-groups-of-drones">4.2.2.2. Groups of drones</a></h4>
<h2 id="43-snp-and-indel-detection"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/">4.3. SNP and indel detection</a></h2>
<h3 id="431-mapping-reads-with-bwa-mem-from-fastq-files-to-bam-files"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#431-mapping-reads-with-bwa-mem-from-fastq-files-to-bam-files">4.3.1. Mapping reads with BWA-MEM: From FASTQ files to BAM files</a></h3>
<h3 id="432-marking-duplicate-reads-with-picard"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#432-marking-duplicate-reads-with-picard">4.3.2. Marking duplicate reads with Picard</a></h3>
<h3 id="433-base-quality-score-recalibration-bqsr-with-gatk"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#433-base-quality-score-recalibration-bqsr-with-gatk">4.3.3. Base quality score recalibration (BQSR) with GATK</a></h3>
<h3 id="434-calling-variants-with-gatk"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#434-calling-variants-with-gatk">4.3.4. Calling variants with GATK</a></h3>
<h3 id="435-combining-all-samples-and-genotypes-with-gatk"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#435-combining-all-samples-and-genotypes-with-gatk">4.3.5. Combining all samples and genotypes with GATK</a></h3>
<h3 id="436-filtering-variants-with-gatk-technical-filters"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#436-filtering-variants-with-gatk-technical-filters">4.3.6. Filtering variants with GATK: Technical filters</a></h3>
<h3 id="437-filtering-variants-with-vcftools-data-quality"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#437-filtering-variants-with-vcftools-data-quality">4.3.7. Filtering variants with VCFtools: Data quality</a></h3>
<h3 id="438-genotype-phasing"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#438-genotype-phasing">4.3.8. Genotype phasing</a></h3>
<h3 id="439-snp-annotation-with-snpeff"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#439-snp-annotation-with-snpeff">4.3.9. SNP annotation with SnpEff</a></h3>
<h3 id="4310-snp-analysis-by-sequencing-pooled-samples"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#4310-snp-analysis-by-sequencing-pooled-samples">4.3.10. SNP analysis by sequencing pooled samples</a></h3>
<h4 id="43101-from-fastq-files-to-bam-files"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#43101-from-fastq-files-to-bam-files">4.3.10.1. From FASTQ files to BAM files</a></h4>
<h4 id="43102-snp-selection-and-pileup-files"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#43102-snp-selection-and-pileup-files">4.3.10.2. SNP selection and pileup files</a></h4>
<h4 id="43103-counting-reads-per-allele-with-popoolation"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#43103-counting-reads-per-allele-with-popoolation">4.3.10.3. Counting reads per allele with PoPoolation</a></h4>
<h2 id="44-comparing-whole-genomes"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_4/">4.4. Comparing whole genomes</a></h2>
<h3 id="441-conducting-a-pairwise-genome-comparison-with-last"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_4/#441-conducting-a-pairwise-genome-comparison-with-last">4.4.1. Conducting a pairwise genome comparison with LAST</a></h3>
<h2 id="45-genome-wide-association-studies"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/">4.5. Genome-wide association studies</a></h2>
<h3 id="451-considerations-for-phenotypic-data"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#451-considerations-for-phenotypic-data">4.5.1. Considerations for phenotypic data</a></h3>
<h3 id="452-considerations-for-sample-selection"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#452-considerations-for-sample-selection">4.5.2. Considerations for sample selection</a></h3>
<h4 id="4521-power-analysis"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#4521-power-analysis">4.5.2.1. Power analysis</a></h4>
<h3 id="453-materials"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#453-materials">4.5.3. Materials</a></h3>
<h4 id="4531-computational-resources"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#4531-computational-resources">4.5.3.1. Computational resources</a></h4>
<h4 id="4532-genotypic-and-phenotypic-data"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#4532-genotypic-and-phenotypic-data">4.5.3.2. Genotypic and phenotypic data</a></h4>
<h3 id="454-methods"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#454-methods">4.5.4. Methods</a></h3>
<h4 id="4541-preparation-of-phenotypic-data"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#4541-preparation-of-phenotypic-data">4.5.4.1. Preparation of phenotypic data</a></h4>
<h4 id="4542-preparation-of-genotypic-data"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#4542-preparation-of-genotypic-data">4.5.4.2. Preparation of genotypic data</a></h4>
<h4 id="4543-performing-gwas-methods-and-software"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#4543-performing-gwas-methods-and-software">4.5.4.3. Performing GWAS: Methods and software</a></h4>
<h4 id="4544-detecting-signatures-of-selection"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#4544-detecting-signatures-of-selection">4.5.4.4. Detecting signatures of selection</a></h4>
<h3 id="455-sources-of-variation"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#455-sources-of-variation">4.5.5. Sources of variation</a></h3>
<h3 id="456-quality-control-and-data-interpretation"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#456-quality-control-and-data-interpretation">4.5.6. Quality control and data interpretation</a></h3>
<h3 id="457-applications-and-limitations"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#457-applications-and-limitations">4.5.7. Applications and limitations</a></h3>
<h2 id="46-population-genomics-experimental-design"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_6/">4.6. Population genomics: Experimental design</a></h2>
<h3 id="461-sampling-strategy"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_6/#461-sampling-strategy">4.6.1. Sampling strategy</a></h3>
<h4 id="4611-sample-sizes-of-individuals-and-markers"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_6/#4611-sample-sizes-of-individuals-and-markers">4.6.1.1. Sample sizes of individuals and markers</a></h4>
<h4 id="4612-sample-breadth"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_6/#4612-sample-breadth">4.6.1.2. Sample breadth</a></h4>
<h4 id="4613-sampling-design"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_6/#4613-sampling-design">4.6.1.3. Sampling design</a></h4>
<h4 id="4614-sampling-workers-versus-drones"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_6/#4614-sampling-workers-versus-drones">4.6.1.4. Sampling workers versus drones</a></h4>
<h4 id="4615-sampling-a-single-individual-versus-multiple-individuals-per-colony"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_6/#4615-sampling-a-single-individual-versus-multiple-individuals-per-colony">4.6.1.5. Sampling a single individual versus multiple individuals per colony</a></h4>
<h2 id="47-population-genomics-filtering-and-summary-statistics-using-plink"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_7/">4.7. Population genomics: Filtering and summary statistics using PLINK</a></h2>
<h3 id="471-download-and-installation"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_7/#471-download-and-installation">4.7.1. Download and installation</a></h3>
<h3 id="472-input-format-and-conversion"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_7/#472-input-format-and-conversion">4.7.2. Input format and conversion</a></h3>
<h4 id="4721-variant-call-format-vcf"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_7/#4721-variant-call-format-vcf">4.7.2.1. Variant call format (VCF)</a></h4>
<h4 id="4722-plink-1-binary-format-bim"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_7/#4722-plink-1-binary-format-bim">4.7.2.2. PLINK 1 binary format (.bim)</a></h4>
<h4 id="4723-regular-plink-text-files"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_7/#4723-regular-plink-text-files">4.7.2.3. Regular PLINK text files</a></h4>
<h4 id="4724-filtering-and-handling-missing-data"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_7/#4724-filtering-and-handling-missing-data">4.7.2.4. Filtering and handling missing data</a></h4>
<h4 id="4725-computing-and-filtering-based-on-allele-frequency"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_7/#4725-computing-and-filtering-based-on-allele-frequency">4.7.2.5. Computing and filtering based on allele frequency</a></h4>
<h4 id="4726-computing-differentiation-indices-wrights-fst"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_7/#4726-computing-differentiation-indices-wrights-fst">4.7.2.6. Computing differentiation indices: Wright's FST</a></h4>
<h4 id="4727-estimating-linkage-disequilibrium"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_7/#4727-estimating-linkage-disequilibrium">4.7.2.7. Estimating linkage disequilibrium</a></h4>
<h2 id="48-population-genomics-inferring-population-structure-using-admixture"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_8/">4.8. Population genomics: Inferring population structure using ADMIXTURE</a></h2>
<h3 id="481-download-and-installation"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_8/#481-download-and-installation">4.8.1. Download and installation</a></h3>
<h3 id="482-input-files"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_8/#482-input-files">4.8.2. Input files</a></h3>
<h3 id="483-methods"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_8/#483-methods">4.8.3. Methods</a></h3>
<h2 id="49-landscape-genomics-an-example-using-lfmm"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_9/">4.9. Landscape genomics: An example using LFMM</a></h2>
<h3 id="491-materials"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_9/#491-materials">4.9.1. Materials</a></h3>
<h3 id="492-methods"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_9/#491-materials">4.9.2. Methods</a></h3>
<h2 id="410-applying-population-genomics-to-conservation-reduced-snp-analysis"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_10/">4.10. Applying population genomics to conservation: Reduced SNP analysis</a></h2>
<h3 id="4101-materials"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_10/#4101-materials">4.10.1. Materials</a></h3>
<h3 id="4102-methods"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_4_10/#4102-methods">4.10.2. Methods</a></h3>
<h1 id="5-epigenomics"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_1/">5. Epigenomics</a></h1>
<h2 id="51-introduction"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_1/">5.1. Introduction</a></h2>
<h2 id="52-dna-methylation"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/">5.2. DNA methylation</a></h2>
<h3 id="521-bisulfite-seq"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/#521-bisulfite-seq">5.2.1. Bisulfite-seq</a></h3>
<h4 id="5211-considerations"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/#5211-considerations">5.2.1.1. Considerations</a></h4>
<h4 id="5212-materials"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/#5212-materials">5.2.1.2. Materials</a></h4>
<h4 id="5213-methods"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/#5213-methods">5.2.1.3. Methods</a></h4>
<h3 id="522-methylated-dna-immunoprecipitation-sequencing-medip-seq"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/#522-methylated-dna-immunoprecipitation-sequencing-medip-seq">5.2.2. Methylated DNA immunoprecipitation-sequencing (MeDIP-seq)</a></h3>
<h4 id="5221-considerations"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/#5221-considerations">5.2.2.1. Considerations</a></h4>
<h4 id="5222-materials"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/#5222-materials">5.2.2.2. Materials</a></h4>
<h4 id="5223-methods"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/#5223-methods">5.2.2.3. Methods</a></h4>
<h3 id="523-data-processing-and-analysis"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/#523-data-processing-and-analysis">5.2.3. Data processing and analysis</a></h3>
<h4 id="5231-software-recommendations"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/#5231-software-recommendations">5.2.3.1. Software recommendations</a></h4>
<h4 id="5232-data-repository"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/#5232-data-repository">5.2.3.2. Data repository</a></h4>
<h4 id="5233-statistical-analysis"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/#5233-statistical-analysis">5.2.3.3. Statistical analysis</a></h4>
<h2 id="53-epitranscriptomics-rna-methylation-of-m6a"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_3/">5.3. Epitranscriptomics: RNA methylation of m6A</a></h2>
<h3 id="531-considerations-for-testing-global-rna-methylation-of-m6a"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_3/#531-considerations-for-testing-global-rna-methylation-of-m6a">5.3.1. Considerations for testing global RNA methylation of m6A</a></h3>
<h3 id="532-materials"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_3/#532-materials">5.3.2. Materials</a></h3>
<h3 id="533-procedure"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_3/#533-procedure">5.3.3. Procedure</a></h3>
<h3 id="534-identifying-methylation-sites"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_3/#534-identifying-methylation-sites">5.3.4. Identifying methylation sites</a></h3>
<h3 id="535-software-recommendations"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_3/#535-software-recommendations">5.3.5. Software recommendations</a></h3>
<h2 id="54-chromatin-organization-and-histone-modifications"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_4/">5.4. Chromatin organization and histone modifications</a></h2>
<h3 id="541-chromatin-immunoprecipitation-sequencing-and-transcription-factor-binding-motifs"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_4/#541-chromatin-immunoprecipitation-sequencing-and-transcription-factor-binding-motifs">5.4.1. Chromatin immunoprecipitation sequencing and transcription factor binding motifs</a></h3>
<h3 id="542-hi-c-chromatin-conformation"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_4/#542-hi-c-chromatin-conformation">5.4.2. Hi-C &amp; chromatin conformation</a></h3>
<h3 id="543-chromatin-accessibility-and-transcriptional-factor-motifs"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_4/#543-chromatin-accessibility-and-transcriptional-factor-motifs">5.4.3. Chromatin accessibility and transcriptional factor motifs</a></h3>
<h3 id="544-detecting-histone-modifications-by-mass-spectrometry"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_4/#544-detecting-histone-modifications-by-mass-spectrometry">5.4.4. Detecting histone modifications by mass spectrometry</a></h3>
<h2 id="55-applications-and-limitations"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_5_5/">5.5. Applications and limitations</a></h2>
<h1 id="6-transcriptomics"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_6_1/">6. Transcriptomics</a></h1>
<h2 id="61-introduction"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_6_1/">6.1. Introduction</a></h2>
<h2 id="62-sequencing-technologies"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_6_2/">6.2. Sequencing technologies</a></h2>
<h3 id="621-considerations"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_6_2/#621-considerations">6.2.1. Considerations</a></h3>
<h3 id="622-illumina-sequencing-short-reads"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_6_2/#622-illumina-sequencing-short-reads">6.2.2. Illumina sequencing (short reads)</a></h3>
<h3 id="623-third-generation-sequencing-long-reads"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_6_2/#623-third-generation-sequencing-long-reads">6.2.3. Third generation sequencing (long reads)</a></h3>
<h4 id="6231-considerations-for-choosing-a-long-read-platform"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_6_2/#6231-considerations-for-choosing-a-long-read-platform">6.2.3.1. Considerations for choosing a long-read platform</a></h4>
<h2 id="63-single-cell-transcriptomics"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_6_3/">6.3. Single-cell transcriptomics</a></h2>
<h3 id="631-considerations"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_6_3/#631-considerations">6.3.1. Considerations</a></h3>
<h3 id="632-materials"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_6_3/#632-materials">6.3.2. Materials</a></h3>
<h3 id="633-sample-preparation-procedure-for-single-cell-sequencing"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_6_3/#633-sample-preparation-procedure-for-single-cell-sequencing">6.3.3. Sample preparation procedure for single-cell sequencing</a></h3>
<h2 id="64-data-handling-and-analysis"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_6_4/">6.4. Data handling and analysis</a></h2>
<h3 id="641-rna-seq-and-differentially-expressed-genes-degs"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_6_4/#641-rna-seq-and-differentially-expressed-genes-degs">6.4.1. RNA-seq and differentially expressed genes (DEGs)</a></h3>
<h3 id="642-gene-network-analysis"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_6_4/#642-gene-network-analysis">6.4.2. Gene network analysis</a></h3>
<h3 id="643-single-cell-transcriptomics"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_6_4/#643-single-cell-transcriptomics">6.4.3. Single-cell transcriptomics</a></h3>
<h4 id="6431-10x-genomics-specific-software"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_6_4/#6431-10x-genomics-specific-software">6.4.3.1. 10x Genomics specific software</a></h4>
<h4 id="6432-third-party-software"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_6_4/#6432-third-party-software">6.4.3.2. Third party software</a></h4>
<h2 id="65-applications-and-limitations"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_6_5/">6.5. Applications and limitations</a></h2>
<h1 id="7-functional-genomics-and-xenobiotic-treatment"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_7_1/">7. Functional genomics and xenobiotic treatment</a></h1>
<h2 id="71-introduction"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_7_1/">7.1. Introduction</a></h2>
<h2 id="72-crispr"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_7_2/">7.2. CRISPR</a></h2>
<h3 id="721-considerations"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_7_2/#721-considerations">7.2.1. Considerations</a></h3>
<h3 id="722-materials"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_7_2/#722-materials">7.2.2. Materials</a></h3>
<h3 id="723-methods-for-crisprcas9-gene-editing-of-embryos"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_7_2/#723-methods-for-crisprcas9-gene-editing-of-embryos">7.2.3. Methods for CRISPR/Cas9 gene editing of embryos</a></h3>
<h4 id="7231-generating-cas9-protein"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_7_2/#7231-generating-cas9-protein">7.2.3.1. Generating Cas9 protein</a></h4>
<h4 id="7232-generating-sgrna"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_7_2/#7232-generating-sgrna">7.2.3.2. Generating sgRNA</a></h4>
<h4 id="7233-ribonucleoprotein-assembly"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_7_2/#7233-ribonucleoprotein-assembly">7.2.3.3. Ribonucleoprotein assembly</a></h4>
<h4 id="7234-egg-collection-and-microinjection"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_7_2/#7234-egg-collection-and-microinjection">7.2.3.4. Egg collection and microinjection</a></h4>
<h2 id="73-rna-interference"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_7_3/">7.3. RNA interference</a></h2>
<h3 id="731-rnai-considerations"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_7_3/#731-rnai-considerations">7.3.1. RNAi considerations</a></h3>
<h3 id="732-methods-for-nanoparticle-mediated-rnai"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_7_3/#732-methods-for-nanoparticle-mediated-rnai">7.3.2. Methods for nanoparticle-mediated RNAi</a></h3>
<h4 id="7321-materials"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_7_3/#7321-materials">7.3.2.1. Materials</a></h4>
<h4 id="7322-procedure"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_7_3/#7322-procedure">7.3.2.2. Procedure</a></h4>
<h2 id="74-xenobiotic-treatment"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_7_4/">7.4. Xenobiotic treatment</a></h2>
<h3 id="741-xenobiotic-treatment-considerations"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_7_4/#741-xenobiotic-treatment-considerations">7.4.1 Xenobiotic treatment considerations</a></h3>
<h3 id="742-materials"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_7_4/#742-materials">7.4.2. Materials</a></h3>
<h3 id="743-procedure"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_7_4/#743-procedure">7.4.3 Procedure</a></h3>
<h4 id="7431-thorax-application"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_7_4/#7431-thorax-application">7.4.3.1 Thorax application</a></h4>
<h4 id="7432-injection"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_7_4/#7432-injection">7.4.3.2. Injection</a></h4>
<h4 id="7433-feeding-individual-bees"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_7_4/#7433-feeding-individual-bees">7.4.3.3. Feeding individual bees</a></h4>
<h4 id="7434-flight-cage-feeding"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_7_4/#7434-flight-cage-feeding">7.4.3.4. Flight cage feeding</a></h4>
<h2 id="75-applications-and-limitations"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_7_5/">7.5. Applications and limitations</a></h2>
<h1 id="8-proteomics"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_8_1/">8. Proteomics</a></h1>
<h2 id="81-introduction"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_8_1/">8.1. Introduction</a></h2>
<h2 id="82-standard-methods-for-shot-gun-proteomics-sample-preparation"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_8_2/">8.2. Standard methods for shot-gun proteomics sample preparation</a></h2>
<h3 id="821-considerations"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_8_2/#821-considerations">8.2.1. Considerations</a></h3>
<h4 id="8211-general"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_8_2/#8211-general">8.2.1.1. General</a></h4>
<h4 id="8212-sample-handling"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_8_2/#8212-sample-handling">8.2.1.2. Sample handling</a></h4>
<h4 id="8213-reagent-handling"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_8_2/#8213-reagent-handling">8.2.1.3. Reagent handling</a></h4>
<h3 id="822-materials"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_8_2/#822-materials">8.2.2. Materials</a></h3>
<h3 id="823-proteomics-methods"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_8_2/#823-proteomics-methods">8.2.3. Proteomics methods</a></h3>
<h4 id="8231-lysis-and-precipitation"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_8_2/#8231-lysis-and-precipitation">8.2.3.1. Lysis and precipitation</a></h4>
<h4 id="8232-solubilization-and-digestion"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_8_2/#8232-solubilization-and-digestion">8.2.3.2. Solubilization and digestion</a></h4>
<h4 id="8233-peptide-desalting-and-resuspension"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_8_2/#8233-peptide-desalting-and-resuspension">8.2.3.3. Peptide desalting and resuspension</a></h4>
<h2 id="83-liquid-chromatography-and-mass-spectrometry"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_8_3/">8.3. Liquid chromatography and mass spectrometry</a></h2>
<h2 id="84-proteomics-data-processing"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_8_4/">8.4. Proteomics data processing</a></h2>
<h3 id="841-software-recommendations"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_8_4/#841-software-recommendations">8.4.1. Software recommendations</a></h3>
<h4 id="8411-maxquant-and-dia-nn-search-parameters"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_8_4/#8411-maxquant-and-dia-nn-search-parameters">8.4.1.1. MaxQuant and DIA-NN search parameters</a></h4>
<h4 id="8412-choosing-an-appropriate-protein-database"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_8_4/#8412-choosing-an-appropriate-protein-database">8.4.1.2. Choosing an appropriate protein database</a></h4>
<h4 id="8413-statistical-analysis"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_8_4/#8413-statistical-analysis">8.4.1.3 Statistical analysis</a></h4>
<h2 id="85-applications-and-limitations"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_8_5/">8.5. Applications and limitations</a></h2>
<h1 id="9-metabolomics"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_9_1/">9. Metabolomics</a></h1>
<h2 id="91-introduction"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_9_1/">9.1. Introduction</a></h2>
<h2 id="92-sample-preparation-for-metabolomics"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_9_2/">9.2. Sample preparation for metabolomics</a></h2>
<h3 id="921-considerations"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_9_2/#921-considerations">9.2.1. Considerations</a></h3>
<h4 id="9211-general"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_9_2/#9211-general">9.2.1.1. General</a></h4>
<h4 id="9212-sample-handling"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_9_2/#9212-sample-handling">9.2.1.2. Sample handling</a></h4>
<h4 id="9213-reagent-handling"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_9_2/#9213-reagent-handling">9.2.1.3. Reagent handling</a></h4>
<h4 id="922-materials"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_9_2/#922-materials">9.2.2. Materials</a></h4>
<h3 id="923-metabolomics-methods"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_9_2/#923-metabolomics-methods">9.2.3. Metabolomics methods</a></h3>
<h4 id="9231-sample-homogenization"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_9_2/#9231-sample-homogenization">9.2.3.1. Sample homogenization</a></h4>
<h4 id="9232-extraction"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_9_2/#9232-extraction">9.2.3.2. Extraction</a></h4>
<h2 id="93-chromatography-and-mass-spectrometry"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_9_3/">9.3. Chromatography and mass spectrometry</a></h2>
<h2 id="94-metabolomics-data-processing"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_9_4/">9.4. Metabolomics data processing</a></h2>
<h2 id="95-metabolomics-applications-and-limitations"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_9_5/">9.5. Metabolomics applications and limitations</a></h2>
<h1 id="10-microbiome-analysis"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_1/">10. Microbiome analysis</a></h1>
<h2 id="101-introduction"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_1/">10.1. Introduction</a></h2>
<h2 id="102-sampling-and-dna-extraction"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/">10.2. Sampling and DNA extraction</a></h2>
<h3 id="1021-considerations"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/#1021-considerations">10.2.1. Considerations</a></h3>
<h4 id="10211-general"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/#10211-general">10.2.1.1. General</a></h4>
<h4 id="10212-tissue-sample-handling"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/#10212-tissue-sample-handling">10.2.1.2. Tissue sample handling</a></h4>
<h4 id="10213-hive-material-sample-handling"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/#10213-hive-material-sample-handling">10.2.1.3. Hive material sample handling</a></h4>
<h3 id="1022-protocol-for-tissue-samples"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/#1022-protocol-for-tissue-samples">10.2.2. Protocol for tissue samples</a></h3>
<h4 id="10221-materials"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/#10221-materials">10.2.2.1. Materials</a></h4>
<h4 id="10222-dissection-methods"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/#10222-dissection-methods">10.2.2.2. Dissection methods</a></h4>
<h4 id="10223-dna-extraction-methods"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/#10223-dna-extraction-methods">10.2.2.3. DNA extraction methods</a></h4>
<h3 id="1023-protocol-for-sampling-hive-materials"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/#1023-protocol-for-sampling-hive-materials">10.2.3. Protocol for sampling hive materials</a></h3>
<h4 id="10231-materials"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/#10231-materials">10.2.3.1. Materials</a></h4>
<h4 id="10232-methods-for-bee-bread-sampling-and-dna-extraction"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/#10232-methods-for-bee-bread-sampling-and-dna-extraction">10.2.3.2. Methods for bee bread sampling and DNA extraction</a></h4>
<h4 id="10233-methods-for-hive-entrance-sampling-and-dna-extraction"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/#10233-methods-for-hive-entrance-sampling-and-dna-extraction">10.2.3.3. Methods for hive entrance sampling and DNA extraction</a></h4>
<h2 id="103-amplicon-sequencing"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_3/">10.3. Amplicon sequencing</a></h2>
<h3 id="1031-considerations"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_3/#1031-considerations">10.3.1. Considerations</a></h3>
<h3 id="1032-materials-for-amplicon-sequencing"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_3/#1032-materials-for-amplicon-sequencing">10.3.2. Materials for amplicon sequencing</a></h3>
<h3 id="1033-methods-for-amplicon-sequencing"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_3/#1033-methods-for-amplicon-sequencing">10.3.3. Methods for amplicon sequencing</a></h3>
<h2 id="104-microbiome-data-analysis"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_4/">10.4. Microbiome data analysis</a></h2>
<h3 id="1041-recommended-software"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_4/#1041-recommended-software">10.4.1. Recommended software</a></h3>
<h3 id="1042-guidance-on-the-data-analysis-methods-an-example-with-qiime-2"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_4/#1042-guidance-on-the-data-analysis-methods-an-example-with-qiime-2">10.4.2. Guidance on the data analysis methods: An example with QIIME 2</a></h3>
<h4 id="10421-importing-data"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_4/#10421-importing-data">10.4.2.1. Importing data</a></h4>
<h4 id="10422-non-biological-sequence-removal"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_4/#10422-non-biological-sequence-removal">10.4.2.2. Non-biological sequence removal</a></h4>
<h4 id="10423-sequence-quality-control-denoising"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_4/#10423-sequence-quality-control-denoising">10.4.2.3. Sequence quality control (denoising)</a></h4>
<h4 id="10424-removing-biological-contamination"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_4/#10424-removing-biological-contamination">10.4.2.4. Removing biological contamination</a></h4>
<h2 id="105-applications-and-limitations"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_10_5/">10.5. Applications and limitations</a></h2>
<h1 id="11-data-management-and-open-access-sharing"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_11_1/">11. Data management and open access sharing</a></h1>
<h2 id="111-metadata-standardization"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_11_1/">11.1. Metadata standardization</a></h2>
<h3 id="1111-common-problems-with-apis-related-bioprojects"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_11_1/#1111-common-problems-with-apis-related-bioprojects">11.1.1. Common problems with Apis-related BioProjects</a></h3>
<h3 id="1112-common-problems-with-apis-related-biosamples"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_11_1/#1112-common-problems-with-apis-related-biosamples">11.1.2. Common problems with Apis-related BioSamples</a></h3>
<h2 id="112-sharing-pipelines-and-scripts"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_11_2/">11.2. Sharing pipelines and scripts</a></h2>
<h1 id="12-the-future-of-apis-omics-biological-integration"><a href="https://maevatecher.github.io/standard-methods-apis-omics/Section_12_1/">12. The future of Apis omics: Biological integration</a></h1>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": [], "search": "../assets/javascripts/workers/search.f8cc74c7.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.c8b220af.min.js"></script>
      
    
  </body>
</html>