
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../about/">
      
      
        <link rel="next" href="../chapt_layout/">
      
      
      <link rel="icon" href="../assets/Coloss-logo.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.7">
    
    
      
        <title>Meet the team - wikiCOLOSS: Standard Methods for Apis Omics Research</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.8608ea7d.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Merriweather+Sans:300,300i,400,400i,700,700i%7CRed+Hat+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Merriweather Sans";--md-code-font:"Red Hat Mono"}</style>
      
    
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="slate" data-md-color-primary="yellow" data-md-color-accent="teal">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#_1" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="wikiCOLOSS: Standard Methods for Apis Omics Research" class="md-header__button md-logo" aria-label="wikiCOLOSS: Standard Methods for Apis Omics Research" data-md-component="logo">
      
  <img src="../assets/Coloss-logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            wikiCOLOSS: Standard Methods for Apis Omics Research
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Meet the team
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="yellow" data-md-color-accent="teal"  aria-label="Dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 7a5 5 0 0 1 5 5 5 5 0 0 1-5 5 5 5 0 0 1-5-5 5 5 0 0 1 5-5m0 2a3 3 0 0 0-3 3 3 3 0 0 0 3 3 3 3 0 0 0 3-3 3 3 0 0 0-3-3m0-7 2.39 3.42C13.65 5.15 12.84 5 12 5s-1.65.15-2.39.42zM3.34 7l4.16-.35A7.2 7.2 0 0 0 5.94 8.5c-.44.74-.69 1.5-.83 2.29zm.02 10 1.76-3.77a7.131 7.131 0 0 0 2.38 4.14zM20.65 7l-1.77 3.79a7.02 7.02 0 0 0-2.38-4.15zm-.01 10-4.14.36c.59-.51 1.12-1.14 1.54-1.86.42-.73.69-1.5.83-2.29zM12 22l-2.41-3.44c.74.27 1.55.44 2.41.44.82 0 1.63-.17 2.37-.44z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="amber" data-md-color-accent="blue"  aria-label="Light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="m17.75 4.09-2.53 1.94.91 3.06-2.63-1.81-2.63 1.81.91-3.06-2.53-1.94L12.44 4l1.06-3 1.06 3zm3.5 6.91-1.64 1.25.59 1.98-1.7-1.17-1.7 1.17.59-1.98L15.75 11l2.06-.05L18.5 9l.69 1.95zm-2.28 4.95c.83-.08 1.72 1.1 1.19 1.85-.32.45-.66.87-1.08 1.27C15.17 23 8.84 23 4.94 19.07c-3.91-3.9-3.91-10.24 0-14.14.4-.4.82-.76 1.27-1.08.75-.53 1.93.36 1.85 1.19-.27 2.86.69 5.83 2.89 8.02a9.96 9.96 0 0 0 8.02 2.89m-1.64 2.02a12.08 12.08 0 0 1-7.8-3.47c-2.17-2.19-3.33-5-3.49-7.82-2.81 3.14-2.7 7.96.31 10.98 3.02 3.01 7.84 3.12 10.98.31"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="wikiCOLOSS: Standard Methods for Apis Omics Research" class="md-nav__button md-logo" aria-label="wikiCOLOSS: Standard Methods for Apis Omics Research" data-md-component="logo">
      
  <img src="../assets/Coloss-logo.png" alt="logo">

    </a>
    wikiCOLOSS: Standard Methods for Apis Omics Research
  </label>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" checked>
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    About
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            About
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../about/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Why a wikiCOLOSS?
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Meet the team
    
  </span>
  

      </a>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../chapt_layout/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Chapter Layout
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    1. The omics revolution in Apis: more data than meets the eye
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    2. Sample management
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    3. Genome sequencing
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            3. Genome sequencing
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_3_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    3.1. Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_3_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    3.2. Genome sequencing technologies
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_3_3/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    3.3. The reference genome
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_3_4/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    3.4. Small and large variant detection
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_3_5/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    3.5. Sequencing museum specimens
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6" >
        
          
          <label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    4. Whole-genome population and association studies
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_6">
            <span class="md-nav__icon md-icon"></span>
            4. Whole-genome population and association studies
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.1. Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.2. Ploidy and sampling considerations
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_3/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.3. SNP and indel detection
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_4/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.4. Comparing whole genomes
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_5/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.5. Genome-wide association studies
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_6/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.6. Population genomics: Experimental design
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_7/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.7. Population genomics: Filtering and summary statistics using PLINK
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_8/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.8. Population genomics: Inferring population structure using ADMIXTURE
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_9/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.9. Landscape genomics: An example using LFMM
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_4_10/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    4.10. Applying population genomics to conservation: Reduced SNP analysis
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_7" >
        
          
          <label class="md-nav__link" for="__nav_7" id="__nav_7_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    5. Epigenomics
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_7_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_7">
            <span class="md-nav__icon md-icon"></span>
            5. Epigenomics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_5_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    5.1. Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_5_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    5.2. DNA methylation
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_5_3/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    5.3. Epitranscriptomics: RNA methylation of m6A
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_5_4/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    5.4. Chromatin organization and histone modifications
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_5_5/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    5.5. Applications and limitations
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_8" >
        
          
          <label class="md-nav__link" for="__nav_8" id="__nav_8_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    6. Transcriptomics
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_8_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_8">
            <span class="md-nav__icon md-icon"></span>
            6. Transcriptomics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_6_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    6.1. Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_6_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    6.2. Sequencing technologies
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_6_3/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    6.3. Single-cell transcriptomics
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_6_4/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    6.4. Data handling and analysis
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_6_5/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    6.5. Applications and limitations
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_9" >
        
          
          <label class="md-nav__link" for="__nav_9" id="__nav_9_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    7. Functional genomics and xenobiotic treatment
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_9_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_9">
            <span class="md-nav__icon md-icon"></span>
            7. Functional genomics and xenobiotic treatment
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_7_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    7.1. Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_7_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    7.2. CRISPR
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_7_3/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    7.3. RNA interference
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_7_4/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    7.4. Xenobiotic treatment
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_7_5/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    7.5. Applications and limitations
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_10" >
        
          
          <label class="md-nav__link" for="__nav_10" id="__nav_10_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    8. Proteomics
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_10_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_10">
            <span class="md-nav__icon md-icon"></span>
            8. Proteomics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_8_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    8.1. Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_8_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    8.2. Standard methods for shot-gun proteomics sample preparation
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_8_3/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    8.3. Liquid chromatography and mass spectrometry
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_8_4/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    8.4. Proteomics data processing
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_8_5/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    8.5. Applications and limitations
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_11" >
        
          
          <label class="md-nav__link" for="__nav_11" id="__nav_11_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    9. Metabolomics
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_11_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_11">
            <span class="md-nav__icon md-icon"></span>
            9. Metabolomics
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_9_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    9.1. Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_9_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    9.2. Sample preparation for metabolomics
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_9_3/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    9.3. Chromatography and mass spectrometry
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_9_4/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    9.4. Metabolomics data processing
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_9_5/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    9.5. Metabolomics applications and limitations
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_12" >
        
          
          <label class="md-nav__link" for="__nav_12" id="__nav_12_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    10. Microbiome analysis
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_12_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_12">
            <span class="md-nav__icon md-icon"></span>
            10. Microbiome analysis
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_10_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    10.1. Introduction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_10_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    10.2. Sampling and DNA extraction
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_10_3/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    10.3. Amplicon sequencing
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_10_4/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    10.4. Microbiome data analysis
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_10_5/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    10.5. Applications and limitations
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_13" >
        
          
          <label class="md-nav__link" for="__nav_13" id="__nav_13_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    11. Data management and open access sharing
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_13_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_13">
            <span class="md-nav__icon md-icon"></span>
            11. Data management and open access sharing
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_11_1/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    11.1. Metadata standardization
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_11_2/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    11.2. Sharing pipelines and scripts
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Section_12/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    12. The future of Apis omics: Biological integration
    
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  



<h1 id="_1"></h1>
<p><img alt="Multi Omics" src="../assets/Multi.png" /></p>
<h1 id="coloss-editor">COLOSS Editor:</h1>
<h2 id="jay-d-evans">Jay D. Evans</h2>
<p>USDA-ARS, Bee Research Lab, BARC-E Bldg 306, Beltsville MD 20705, USA.</p>
<p>[Short Bio: -]</p>
<h1 id="authors-short-bio-by-alphabetical-order">Authors short bio (by alphabetical order):</h1>
<p><img alt="Priyadarshini Chakrabarti" src="../assets/Priya.jpg" /></p>
<h2 id="priyadarshini-chakrabarti">Priyadarshini Chakrabarti</h2>
<p>Washington State University affiliation Department of Entomology, Washington State University, Pullman, WA 99164, USA</p>
<h3 id="expertise-metabolomics-proteomics-lipidomics-apis-mellifera">Expertise: Metabolomics, proteomics, lipidomics, <em>Apis mellifera</em></h3>
<p>Priya is an Assistant Professor at the Department of Entomology at Washington State University. Priya is also a courtesy faculty at Oregon State University. Priya studies the interactive impacts of multiple stressors on bees, for example poor nutrition, pesticides, and climate change. She uses a wide array of multidisciplinary techniques across fields such as physiology, toxicology, functional biology, multi-omics approaches, and neuroethology to address her research questions. She is currently serving as the secretary/treasurer of the American Association of Professional Apiculturists, the North American Chair of the COLOSS bee nutrition taskforce and is the Vice-President Elect of the PBT Section of the Entomological Society of America. She also served as the chair of the Early Career Professionals Committee of the Entomological Society of America. In addition to the research community, Priya enjoys working with stakeholders, policymakers and the general community in protecting bee pollinators and raising pollinator awareness. She is also a children's book author who engages young readers and helps protect pollinators. More information about her research can be found on her lab’s website.</p>
<p><img alt="Lílian Caesar" src="../assets/Lilian.jpg" /></p>
<h2 id="lilian-caesar">Lílian Caesar</h2>
<p>Department of Biology, Indiana University, Bloomington, Indiana, USA</p>
<h3 id="expertise-omics-microbiomes-ecology-and-evolution-of-symbiosis-host-microbe-and-microbe-microbe-interactions-honey-bees-and-stingless-bees">Expertise: Omics, microbiomes, ecology and evolution of symbiosis, host-microbe and microbe-microbe interactions, honey bees, and stingless bees</h3>
<p>Lílian earned her PhD at the Universidade Federal do Rio Grande do Sul, Brazil. Currently she is a postdoctoral fellow in the Newton Lab, Department of Biology, at Indiana University, Bloomington. Her research focuses on the eco-evolutionary dynamics of symbiosis, and the impact of symbionts for host health. Her model system is the highly eusocial bees, i.e. honey bees and stingless bees. Lílian is interested in characterizing and studying microbiomes, including bacteria, fungi and viruses associated both with the adult bees, the brood in development and hive environments. For her investigations, Lílian combines multi-omics, in vitro/in vivo microbiology assays, and field work.</p>
<p><img alt="Sonia E. Eynard" src="../assets/Sonia.jpg" /></p>
<h2 id="sonia-e-eynard">Sonia E. Eynard</h2>
<p>GenPhySE, Université de Toulouse, INRAE, ENVT, 31326, Castanet Tolosan, France</p>
<h3 id="expertise-population-genomics-quantitative-genetics-livestock-species-apis-mellifera">Expertise: Population genomics, quantitative genetics, livestock species, <em>Apis mellifera</em></h3>
<p>As an undergraduate, Sonia studied wildlife conservation and the use of population genetics to advise on conservation strategies. In 2018 she graduated with a PhD in Animal Breeding and Genetics from the Erasmus Mundus project EGS-ABG with a double diploma from Wageningen University and AgroParisTech, where she looked into ways to mitigate the impact of selection on genetic diversity in livestock using next generation sequencing data. After completing her PhD she joined INRAE, in Toulouse, to work on the BeeStrong project. The aim was to identify genetic markers linked to resistance to <em>Varroa destructor</em> infestation in the French <em>Apis mellifera</em> population using pool sequencing experiments. She also contributed to a population genomics project to better understand and describe the genetic diversity observed in the European bee populations. Always fond of analysing new data and developing new methods, in 2023 she moved to a new postdoctoral position working on epigenetics and how it can be included in current breeding decisions in livestock species. Very much in love with honey bees, she hopes to be able to develop her own research line using new data and methods to answer challenges faced in honey bee genomics.</p>
<p><img alt="Catherine Farrell" src="../assets/Catherine.jpg" /></p>
<h2 id="m-catherine-farrell">M. Catherine Farrell</h2>
<p>Agricultural Research and Development Program, Central State University, Wilberforce, OH</p>
<h3 id="expertise-behavior-single-cell-transcriptomics-neurobiology-apis-mellifera">Expertise: Behavior, single cell transcriptomics, neurobiology, <em>Apis mellifera</em></h3>
<p>Catherine is a Research Assistant Professor of bee genomics at Central State University. As an undergraduate, her primary research interests were in the transcriptional regulation of circadian clock genes in <em>Xenopus</em> frogs. As a graduate student, her dissertation research focused on <em>Drosophila</em> neurobiology. In particular, she studied the role of serotonin in sensorimotor plasticity, and applied the diverse tools available within the <em>Drosophila</em> model system to look specifically at the neural circuitry underlying coordinated crawling behavior. After taking some time off after gaining her PhD in 2015 to teach, she joined Central State as a postdoctoral fellow, where she utilized transcriptomics, single cell transcriptomics, and epigenomics to understand honey bee grooming behavior.</p>
<p><img alt="Leonard J. Foster" src="../assets/Leonard.jpg" /></p>
<h2 id="leonard-j-foster">Leonard J. Foster</h2>
<p>Department of Biochemistry and Molecular Biology, Michael Smith Laboratories, University of British Columbia, Vancouver, BC V6T1Z4, Canada</p>
<h3 id="expertise-proteomics-bioinformatics-apis-mellifera-and-many-other-model-systems">Expertise: Proteomics, bioinformatics, <em>Apis mellifera</em> and many other model systems</h3>
<p>Leonard is a Professor in the Department of Biochemistry and Molecular Biology at the University of British Columbia. Dr. Foster comes from a family of beekeepers and got his introduction to academic bee research while doing his Bachelor’s degree in biochemistry at Simon Fraser University, where he worked with Drs. Winston and Slessor on honey bee pheromones, particularly the components of queen mandibular pheromone. He then completed a PhD in Toronto and a postdoctoral fellowship in Denmark before starting his current position in 2005. The first independent operating grant that Dr. Foster secured was to study how bee pathogens were able to manipulate the protein machinery within bee cells. Since that time he has led three very large-scale projects that have investigated some of the molecular mechanisms behind disease resistance in bees. This effort has recently moved into trying to apply this knowledge by using this information to guide selective breeding for hygienic behavior in honey bees. He is very active in extension and frequently engages the public on various aspects of honey bee biology.</p>
<p><img alt="June Gorrochategui-Ortega" src="../assets/June.jpg" /></p>
<h2 id="june-gorrochategui-ortega">June Gorrochategui-Ortega</h2>
<p>Department of Genetics, Physical Anthropology and Animal Physiology, Applied Genomics and Bioinformatics lab., University of the Basque Country (UPV/EHU), Leioa, Spain</p>
<h3 id="expertise-bioinformatics-metagenomics-apis-mellifera">Expertise: Bioinformatics, metagenomics, <em>Apis mellifera</em></h3>
<p>June is a PhD student at the Applied Genomics and Bioinformatics group, at the Department of Genetics, Physical Anthropology and Animal Physiology of the University of the Basque Country (UPV/EHU, Spain). She is currently working on her PhD thesis (Basque Government PhD student grant), entitled “Identification of microbial profiles to promote colony health: towards the reduction of chemical residues in Apiculture (ERLEMIKRO)”. Her research aims to identify and characterize the microbial taxonomic and functional diversity of honey bee hive, for the identification of profiles that harbor symbiotic properties beneficial for honey bee health; by using metagenomics and metatranscriptomics.</p>
<p><img alt="Dora Henriques" src="../assets/Dora.jpg" /></p>
<h2 id="dora-henriques">Dora Henriques</h2>
<p>CIMO, LA SusTEC, Instituto Politécnico de Bragança, Campus de Santa Apolónia, 5300-253 Bragança, Portugal.</p>
<h3 id="expertise-population-genomics-bioinformatics-conservation-genomics-apis-mellifera">Expertise: Population genomics, bioinformatics, conservation genomics, <em>Apis mellifera</em></h3>
<p>Dora is a junior researcher at the Mountain Research Centre (CIMO) and at the Associated Laboratory for Sustainability and Technology in Mountain Regions (SusTEC). She completed her PhD in Molecular and Environmental Biology at Minho University in 2018 with the thesis entitled “Employment of whole genome resequencing to reveal the evolutionary history and to develop molecular tools for Western European honey bees (<em>Apis mellifera</em> subspecies)”. Her main research interests are honey bee evolution and the development of molecular tools for honey bee conservation. She has collaborated on eight research projects, publishing over 20 peer-reviewed scientific articles and 15 popular articles, while also presenting over 90 conference papers. She is a member of the editorial board of the Journal of Apicultural Research (JAR).</p>
<p><img alt="Hongmei Li-Byarlay" src="../assets/Hongmei.jpg" /></p>
<h2 id="hongmei-li-byarlay">Hongmei Li-Byarlay</h2>
<p>Agricultural Research and Development Program, Central State University, Wilberforce, OH Department of Agricultural and Life Sciences, Central State University, Wilberforce, OH</p>
<h3 id="expertise-transcriptomics-epigenomics-functional-genomics-of-honey-bees">Expertise: Transcriptomics, epigenomics, functional genomics of honey bees</h3>
<p>Hongmei is currently an Associate Professor at Central State University in the State of Ohio (U.S.A). Her doctoral degree is in entomology from Purdue University (Indiana) where she studied the genetics and physiology of Drosophila melanogaster with Drs. Barry Pittendrigh and Larry Murdock. She then worked at the University of Illinois at Urbana-Champaign (UIUC) and at North Carolina State University for her postdoctoral research. Since 2010, she has conducted her research on bee behavior, genetics, molecular biology and physiology. In 2017, she started her own lab’s research in honey bee functional genomics, transcriptomics, epigenomics such as DNA and RNA methylation, molecular mechanisms underlying social behaviors, selection and breeding for mite resistant stocks, oxidative stress and aging. She has published 40+ peer-reviewed papers and book chapters. She has presented 100+ talks for research, education, extension, outreach to different communities. Her research lab received funding support from the National Science Foundation and U.S. Department of Agriculture. More information can be found on her lab’s website and her Google Scholar page.</p>
<p><img alt="Alison McAfee" src="../assets/Alison.jpg" /></p>
<h2 id="alison-mcafee">Alison McAfee</h2>
<p>Department of Biochemistry and Molecular Biology, Michael Smith Laboratories, University of British Columbia, Vancouver, BC V6T1Z4, Canada</p>
<p>Department of Applied Ecology, North Carolina State University, Raleigh, NC 27695-7617, USA</p>
<h3 id="expertise-proteomics-queen-biology-apis-mellifera">Expertise: Proteomics, queen biology, <em>Apis mellifera</em></h3>
<p>Alison completed her PhD in Genome Science and Technology at the University of British Columbia in 2018. She is now a postdoctoral fellow and uses mass spectrometry to study how extreme temperatures associated with climate change are affecting honey bee and bumble bee fertility and survival. Alison also loves science communication, and has published 90+ magazine articles for publications including Scientific American, American Bee Journal, Conversation Canada, and UBC Magazine. Her research has been covered by outlets such as New Scientist, National Geographic, Science Friday and Scientific American, among others. In 2022, Alison received the L’Oreal-UNESCO for Women in Science International Rising Talents award for North America, becoming one of fifteen top female scientists globally who are recognized for their leadership and research each year.</p>
<p><img alt="Jeffrey T. Morré" src="../assets/Jeffrey.jpg" /></p>
<h2 id="jeffrey-t-morre">Jeffrey T. Morré</h2>
<p>Department of Chemistry, Oregon State University, Corvallis, OR 97331 USA</p>
<h3 id="expertise-biological-mass-spectrometry">Expertise: Biological mass spectrometry</h3>
<p>Jeffrey has 26 years of mass spectrometry experience. Jeffrey's current position is operations manager for the Oregon State University Mass Spectrometry Center.</p>
<p><img alt="Irene L. G. Newton " src="../assets/Irene.jpg" /></p>
<h2 id="irene-l-g-newton">Irene L. G. Newton</h2>
<p>Professor of Biology, Department of Biology, Indiana University, Bloomington, Indiana, USA</p>
<h3 id="expertise-host-microbe-interactions-microbial-genomics-microbiomes">Expertise: Host-microbe interactions, microbial genomics, microbiomes</h3>
<p>Irene is a Professor of Biology at Indiana University, Bloomington where she studies mechanisms of host-microbe symbiosis. One of her model systems is the honey bee microbiome, where she focuses on understanding the composition of the community and its function.</p>
<p><img alt="Melanie Parejo" src="../assets/Melanie.jpg" /></p>
<h2 id="melanie-parejo">Melanie Parejo</h2>
<p>Department of Genetics, Physical Anthropology and Animal Physiology, Applied Genomics and Bioinformatics lab., University of the Basque Country (UPV/EHU), Leioa, Spain</p>
<h3 id="expertise-population-genomics-bioinformatics-apis-mellifera">Expertise: Population genomics, bioinformatics, <em>Apis mellifera</em></h3>
<p>Melanie has a life science background, holding a master's degree in Ecology and Evolution from the Swiss Federal Institute of Technology, Zurich, (ETHZ). She also holds a PhD in Biochemistry and Molecular Biology (2017) from the University of Bern, Switzerland, in collaboration with the Swiss Bee Research Center, Agroscope, the topic of her thesis being “Honey bee Conservation and Population Genetics using Whole-Genome Sequence Data”. With a postdoctoral fellowship from the Swiss National Science Foundation, she now works at the University of the Basque Country (UPV/EHU), where she continues to study honey bee genomics. Since 2018, she has been the Assistant Editor of the Journal of Apicultural Research, published on behalf of the International Bee Research Association. Her interests lie in applied genomics in apiculture, diagnostic molecular tools, conservation, ecology and hologenomics. Findings of her research have led to new and adapted management practices such as increasing the hybridization threshold for native honey bees in conservation areas, genetic monitoring tools such as a high-throughput introgression test for bee breeding based on single-nucleotide polymorphisms (SNPs), a subspecies diagnostic SNP chip to classify European subspecies, and a SNP-based paternity assignment test to be employed in the evaluation of breeding mating stations’ efficiency.</p>
<p><img alt="M. Alice Pinto" src="../assets/Alice.jpg" /></p>
<h2 id="m-alice-pinto">M. Alice Pinto</h2>
<p>CIMO, LA SusTEC, Instituto Politécnico de Bragança, Campus de Santa Apolónia, 5300-253 Bragança, Portugal.</p>
<h3 id="expertise-population-genomics-apis-mellifera">Expertise: Population genomics, <em>Apis mellifera</em></h3>
<p>Alice is a Professor at the Polytechnic Institute of Bragança (IPB) and a researcher at the Mountain Research Centre (CIMO), Portugal. She earned a doctoral degree in entomology from Texas A&amp;M University in the United States. She developed a keen interest in honey bees during her PhD research, while studying a fascinating feral population living in oak tree cavities. For her PhD dissertation, she examined the genetic changes in this population undergoing Africanization. Currently, her main research interests involve uncovering the processes that shape extant genetic diversity patterns in honey bee populations, with a particular focus on understanding the basis of local adaptation. She has coordinated and collaborated in over 15 national and international research projects funded by the Portuguese Science Foundation and the European Commission. These projects have dealt with a wide array of subjects, such as: (i) the genomics of <em>Apis mellifera</em> to the genetics of its predators (<em>Vespa velutina</em>) and diseases (viruses and nosemosis), and (ii) the development of SNP-based assays and ML-based software for honey bee identification, and (iii) the botanical identification of bee-collected pollen via ITS2-metabarcoding. She has published over 60 peer-reviewed scientific articles, 40 popular articles, and presented over 250 conference papers. She is a member of the editorial board of Scientific Reports, Insects, and Frontiers in Bee Science.</p>
<p><img alt="Maeva A. Techer" src="../assets/Maeva.jpg" /></p>
<h2 id="maeva-a-techer">Maeva A. Techer</h2>
<p>Okinawa Institute of Science and Technology, 1919-1 Tancha, Onna-son,904-0495 Okinawa, Japan</p>
<p>Texas A&amp;M University, Department of Entomology, College Station, USA</p>
<p>Behavioral Plasticity Research Institute, NSF-BII, USA</p>
<h3 id="expertise-population-genomics-comparative-genomics-transcriptomics-apis-mellifera-varroa-destructor">Expertise: Population Genomics, Comparative genomics, Transcriptomics, <em>Apis mellifera</em>, Varroa destructor</h3>
<p>Maeva joined the Behavioral Plasticity Research Institute as a Postdoctoral Researcher and has been based at Texas A&amp;M University since 2021. While her current research focuses on locust phase polyphenism, she is dedicating herself to learning how to bridge the gap between disciplines and find shared evolution in diverse arthropods to contribute to the new generation of integrative biologists. She completed a PhD in Population Genetics on insular native honey bees evolution at the University of Reunion Island (where she was born). There she learned to combine classical population genetics and demographic inferences to retrace the origins of modern populations and understand which factors shaped their evolution over time. She started using whole genome sequencing during her postdoctoral position at the Okinawa Institute of Science and Technology in Japan, where she focused on the most detrimental honey bee parasites: Varroa mites. Her research seeks to understand how species and populations can rapidly respond and adapt to environmental changes by generating and maintaining variability.</p>
<p><img alt="Alain Vignal" src="../assets/Alain.jpg" /></p>
<h2 id="alain-vignal">Alain Vignal</h2>
<p>GenPhySE, Université de Toulouse, INRAE, ENVT, 31326, Castanet Tolosan, France</p>
<h3 id="expertise-population-and-comparative-genomics-qtl-mapping-poultry-apis-mellifera">Expertise: Population and comparative genomics, QTL mapping, poultry, <em>Apis mellifera</em></h3>
<p>Alain obtained his PhD in molecular biology from Orsay University (France) in 1991. His first expertise was on sequencing genes coding for human blood groups and building genetic maps for locating markers linked to monogenic diseases. He now works at INRAE in Toulouse, France, where he first worked on QTL mapping in chickens and was subsequently involved in the consortiums for the assembly of reference genomes of several poultry species. First the chicken in 2004 (Sanger sequencing) and helping with updates until 2017, then the mallard duck, Muskovy duck, quail, and guinea fowl. With the establishment of short-read parallel sequencing and long-read sequencing as new standards, population genomics at the genome scale became possible, which he did in guinea fowl and now in honey bees. His main focus is to better understand the genetic diversity and admixture in European honey bee populations and to deepen our understanding of the honey bee genome structure.</p>
<p><img alt="Iratxe Zarraonaindia" src="../assets/Iraxte.png" /></p>
<h2 id="iratxe-zarraonaindia">Iratxe Zarraonaindia</h2>
<p>Department of Genetics, Physical Anthropology and Animal Physiology, Applied Genomics and Bioinformatics lab., University of the Basque Country (UPV/EHU), Leioa, Spain IKERBASQUE, Basque Foundation for Science, Bilbao, Spain</p>
<h3 id="expertise-bioinformatics-metagenomics-apis-mellifera_1">Expertise: Bioinformatics, metagenomics, <em>Apis mellifera</em></h3>
<p>Zarraonaindia is an Ikerbasque Research Associate at the Department of Genetics, Physical Anthropology and Animal Physiology of the University of the Basque Country (UPV/EHU, Spain) since 2021. She conducted her postdoctoral research (2012-2015) at the Bioscience Division of Argonne National Laboratory (Chicago, USA), where she acquired ample expertise on environmental microbiology and NGS bioinformatics by joining a massively collaborative project that aimed to characterize microbial life on Earth (The Earth Microbiome Project). In 2016 Zarraonaindia joined the Applied Genomics and Bioinformatics group at UPV/EHU, recently becoming the lead investigator. Her current research interest is implementing NGS approaches for the investigation of the impact of anthropic activities on apiculture, by better understanding microbe-bee interactions and how microbes influence honey bee disease resistance, working towards environmentally friendly alternatives to agrochemicals. She teaches “Bioinformatics and omics” classes at several master’s programs and has participated in 3 European, 1 national and a local research project to unravel the contribution of landscape, environment, management, genetics, and microbiota in honey bee health.</p>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": [], "search": "../assets/javascripts/workers/search.f8cc74c7.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.c8b220af.min.js"></script>
      
    
  </body>
</html>