# [1. The â€˜omics revolution in Apis: More data than meets the eye](https://maevatecher.github.io/standard-methods-apis-omics/Section_1/)

# [2. Sample management](https://maevatecher.github.io/standard-methods-apis-omics/Section_2/)

# [3. Genome sequencing](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_1/)

## [3.1. Introduction](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_1/)

## [3.2. Genome sequencing technologies](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_2/)

### [3.2.1. Sanger sequencing](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_2/#321-sanger-sequencing)

### [3.2.2. Next generation sequencing](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_2/#322-next-generation-sequencing)

### [3.2.3. Long-read sequencing](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_2/#323-long-read-sequencing)

## [3.3. The reference genome](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_3/)

### [3.3.1. Assembling the reference genome](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_3/#331-assembling-the-reference-genome)

#### [3.3.1.1. Hi-C chromosome conformation capture](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_3/#3311-hi-c-chromosome-conformation-capture)

#### [3.3.1.2. DNA source selection](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_3/#3312-dna-source-selection)

### [3.3.2. Annotating the reference genome](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_3/#332-annotating-the-reference-genome)

### [3.3.3. High molecular weight DNA extraction](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_3/#333-high-molecular-weight-dna-extraction)

## [3.4. Small and large variant detection](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_4/)

### [3.4.1. RAD-seq](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_4/#341-rad-seq)

## [3.5. Sequencing museum specimens](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_5/)

### [3.5.1. Considerations](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_5/#351-considerations)

### [3.5.2. Materials](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_5/#352-materials)

### [3.5.3. Procedure](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_5/#353-procedure)

#### [3.5.3.1. Preparation and lysis](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_5/#3531-preparation-and-lysis)

#### [3.5.3.2. DNA extraction](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_5/#3532-dna-extraction)

#### [3.5.3.3. Precipitation](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_5/#3533-precipitation)

#### [3.5.3.4. Washing](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_5/#3534-washing)

#### [3.5.3.5. Final solubilization](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_5/#3535-final-solubilization)

### [3.5.4. Sequencing of museum and ancient genomes](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_5/#354-sequencing-of-museum-and-ancient-genomes)

### [3.5.5. Guidance on the data analysis methods](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_5/#355-guidance-on-the-data-analysis-methods)

### [3.5.6. Applications and limitations](https://maevatecher.github.io/standard-methods-apis-omics/Section_3_5/#356-applications-and-limitations)

# [4. Whole-genome population and association studies](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_1/)

## [4.1. Introduction](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_1/)

## [4.2. Ploidy and sampling considerations](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_2/)

### [4.2.1. Individual sampling](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_2/#421-individual-sampling)

### [4.2.2. Pooled sampling](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_2/#422-pooled-sampling)

#### [4.2.2.1. Groups of workers](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_2/#4221-groups-of-workers)

#### [4.2.2.2. Groups of drones](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_2/#4222-groups-of-drones)

## [4.3. SNP and indel detection](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/)

### [4.3.1. Mapping reads with BWA-MEM: From FASTQ files to BAM files](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#431-mapping-reads-with-bwa-mem-from-fastq-files-to-bam-files)

### [4.3.2. Marking duplicate reads with Picard](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#432-marking-duplicate-reads-with-picard)

### [4.3.3. Base quality score recalibration (BQSR) with GATK](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#433-base-quality-score-recalibration-bqsr-with-gatk)

### [4.3.4. Calling variants with GATK](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#434-calling-variants-with-gatk)

### [4.3.5. Combining all samples and genotypes with GATK](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#435-combining-all-samples-and-genotypes-with-gatk)

### [4.3.6. Filtering variants with GATK: Technical filters](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#436-filtering-variants-with-gatk-technical-filters)

### [4.3.7. Filtering variants with VCFtools: Data quality](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#437-filtering-variants-with-vcftools-data-quality)

### [4.3.8. Genotype phasing](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#438-genotype-phasing)

### [4.3.9. SNP annotation with SnpEff](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#439-snp-annotation-with-snpeff)

### [4.3.10. SNP analysis by sequencing pooled samples](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#4310-snp-analysis-by-sequencing-pooled-samples)

#### [4.3.10.1. From FASTQ files to BAM files](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#43101-from-fastq-files-to-bam-files)

#### [4.3.10.2. SNP selection and pileup files](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#43102-snp-selection-and-pileup-files)

#### [4.3.10.3. Counting reads per allele with PoPoolation](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_3/#43103-counting-reads-per-allele-with-popoolation)

## [4.4. Comparing whole genomes](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_4/)

### [4.4.1. Conducting a pairwise genome comparison with LAST](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_4/#441-conducting-a-pairwise-genome-comparison-with-last)

## [4.5. Genome-wide association studies](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/)

### [4.5.1. Considerations for phenotypic data](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#451-considerations-for-phenotypic-data)

### [4.5.2. Considerations for sample selection](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#452-considerations-for-sample-selection)

#### [4.5.2.1. Power analysis](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#4521-power-analysis)

### [4.5.3. Materials](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#453-materials)

#### [4.5.3.1. Computational resources](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#4531-computational-resources)

#### [4.5.3.2. Genotypic and phenotypic data](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#4532-genotypic-and-phenotypic-data)

### [4.5.4. Methods](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#454-methods)

#### [4.5.4.1. Preparation of phenotypic data](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#4541-preparation-of-phenotypic-data)

#### [4.5.4.2. Preparation of genotypic data](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#4542-preparation-of-genotypic-data)

#### [4.5.4.3. Performing GWAS: Methods and software](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#4543-performing-gwas-methods-and-software)

#### [4.5.4.4. Detecting signatures of selection](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#4544-detecting-signatures-of-selection)

### [4.5.5. Sources of variation](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#455-sources-of-variation)

### [4.5.6. Quality control and data interpretation](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#456-quality-control-and-data-interpretation)

### [4.5.7. Applications and limitations](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_5/#457-applications-and-limitations)

## [4.6. Population genomics: Experimental design](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_6/)

### [4.6.1. Sampling strategy](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_6/#461-sampling-strategy)

#### [4.6.1.1. Sample sizes of individuals and markers](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_6/#4611-sample-sizes-of-individuals-and-markers)

#### [4.6.1.2. Sample breadth](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_6/#4612-sample-breadth)

#### [4.6.1.3. Sampling design](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_6/#4613-sampling-design)

#### [4.6.1.4. Sampling workers versus drones](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_6/#4614-sampling-workers-versus-drones)

#### [4.6.1.5. Sampling a single individual versus multiple individuals per colony](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_6/#4615-sampling-a-single-individual-versus-multiple-individuals-per-colony)

## [4.7. Population genomics: Filtering and summary statistics using PLINK](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_7/)

### [4.7.1. Download and installation](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_7/#471-download-and-installation)

### [4.7.2. Input format and conversion](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_7/#472-input-format-and-conversion)

#### [4.7.2.1. Variant call format (VCF)](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_7/#4721-variant-call-format-vcf)

#### [4.7.2.2. PLINK 1 binary format (.bim)](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_7/#4722-plink-1-binary-format-bim)

#### [4.7.2.3. Regular PLINK text files](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_7/#4723-regular-plink-text-files)

#### [4.7.2.4. Filtering and handling missing data](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_7/#4724-filtering-and-handling-missing-data)

#### [4.7.2.5. Computing and filtering based on allele frequency](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_7/#4725-computing-and-filtering-based-on-allele-frequency)

#### [4.7.2.6. Computing differentiation indices: Wright's FST](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_7/#4726-computing-differentiation-indices-wrights-fst)

#### [4.7.2.7. Estimating linkage disequilibrium](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_7/#4727-estimating-linkage-disequilibrium)

## [4.8. Population genomics: Inferring population structure using ADMIXTURE](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_8/)

### [4.8.1. Download and installation](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_8/#481-download-and-installation)

### [4.8.2. Input files](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_8/#482-input-files)

### [4.8.3. Methods](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_8/#483-methods)

## [4.9. Landscape genomics: An example using LFMM](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_9/)

### [4.9.1. Materials](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_9/#491-materials)

### [4.9.2. Methods](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_9/#491-materials)

## [4.10. Applying population genomics to conservation: Reduced SNP analysis](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_10/)

### [4.10.1. Materials](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_10/#4101-materials)

### [4.10.2. Methods](https://maevatecher.github.io/standard-methods-apis-omics/Section_4_10/#4102-methods)

# [5. Epigenomics](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_1/)

## [5.1. Introduction](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_1/)

## [5.2. DNA methylation](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/)

### [5.2.1. Bisulfite-seq](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/#521-bisulfite-seq)

#### [5.2.1.1. Considerations](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/#5211-considerations)

#### [5.2.1.2. Materials](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/#5212-materials)

#### [5.2.1.3. Methods](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/#5213-methods)

### [5.2.2. Methylated DNA immunoprecipitation-sequencing (MeDIP-seq)](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/#522-methylated-dna-immunoprecipitation-sequencing-medip-seq)

#### [5.2.2.1. Considerations](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/#5221-considerations)

#### [5.2.2.2. Materials](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/#5222-materials)

#### [5.2.2.3. Methods](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/#5223-methods)

### [5.2.3. Data processing and analysis](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/#523-data-processing-and-analysis)

#### [5.2.3.1. Software recommendations](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/#5231-software-recommendations)

#### [5.2.3.2. Data repository](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/#5232-data-repository)

#### [5.2.3.3. Statistical analysis](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_2/#5233-statistical-analysis)

## [5.3. Epitranscriptomics: RNA methylation of m6A](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_3/)

### [5.3.1. Considerations for testing global RNA methylation of m6A](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_3/#531-considerations-for-testing-global-rna-methylation-of-m6a)

### [5.3.2. Materials](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_3/#532-materials)

### [5.3.3. Procedure](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_3/#533-procedure)

### [5.3.4. Identifying methylation sites](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_3/#534-identifying-methylation-sites)

### [5.3.5. Software recommendations](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_3/#535-software-recommendations)

## [5.4. Chromatin organization and histone modifications](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_4/)

### [5.4.1. Chromatin immunoprecipitation sequencing and transcription factor binding motifs](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_4/#541-chromatin-immunoprecipitation-sequencing-and-transcription-factor-binding-motifs)

### [5.4.2. Hi-C & chromatin conformation](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_4/#542-hi-c-chromatin-conformation)

### [5.4.3. Chromatin accessibility and transcriptional factor motifs](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_4/#543-chromatin-accessibility-and-transcriptional-factor-motifs)

### [5.4.4. Detecting histone modifications by mass spectrometry](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_4/#544-detecting-histone-modifications-by-mass-spectrometry)

## [5.5. Applications and limitations](https://maevatecher.github.io/standard-methods-apis-omics/Section_5_5/)

# [6. Transcriptomics](https://maevatecher.github.io/standard-methods-apis-omics/Section_6_1/)

## [6.1. Introduction](https://maevatecher.github.io/standard-methods-apis-omics/Section_6_1/)

## [6.2. Sequencing technologies](https://maevatecher.github.io/standard-methods-apis-omics/Section_6_2/)

### [6.2.1. Considerations](https://maevatecher.github.io/standard-methods-apis-omics/Section_6_2/#621-considerations)

### [6.2.2. Illumina sequencing (short reads)](https://maevatecher.github.io/standard-methods-apis-omics/Section_6_2/#622-illumina-sequencing-short-reads)

### [6.2.3. Third generation sequencing (long reads)](https://maevatecher.github.io/standard-methods-apis-omics/Section_6_2/#623-third-generation-sequencing-long-reads)

#### [6.2.3.1. Considerations for choosing a long-read platform](https://maevatecher.github.io/standard-methods-apis-omics/Section_6_2/#6231-considerations-for-choosing-a-long-read-platform)

## [6.3. Single-cell transcriptomics](https://maevatecher.github.io/standard-methods-apis-omics/Section_6_3/)

### [6.3.1. Considerations](https://maevatecher.github.io/standard-methods-apis-omics/Section_6_3/#631-considerations)

### [6.3.2. Materials](https://maevatecher.github.io/standard-methods-apis-omics/Section_6_3/#632-materials)

### [6.3.3. Sample preparation procedure for single-cell sequencing](https://maevatecher.github.io/standard-methods-apis-omics/Section_6_3/#633-sample-preparation-procedure-for-single-cell-sequencing)

## [6.4. Data handling and analysis](https://maevatecher.github.io/standard-methods-apis-omics/Section_6_4/)

### [6.4.1. RNA-seq and differentially expressed genes (DEGs)](https://maevatecher.github.io/standard-methods-apis-omics/Section_6_4/#641-rna-seq-and-differentially-expressed-genes-degs)

### [6.4.2. Gene network analysis](https://maevatecher.github.io/standard-methods-apis-omics/Section_6_4/#642-gene-network-analysis)

### [6.4.3. Single-cell transcriptomics](https://maevatecher.github.io/standard-methods-apis-omics/Section_6_4/#643-single-cell-transcriptomics)

#### [6.4.3.1. 10x Genomics specific software](https://maevatecher.github.io/standard-methods-apis-omics/Section_6_4/#6431-10x-genomics-specific-software)

#### [6.4.3.2. Third party software](https://maevatecher.github.io/standard-methods-apis-omics/Section_6_4/#6432-third-party-software)

## [6.5. Applications and limitations](https://maevatecher.github.io/standard-methods-apis-omics/Section_6_5/)

# [7. Functional genomics and xenobiotic treatment](https://maevatecher.github.io/standard-methods-apis-omics/Section_7_1/)

## [7.1. Introduction](https://maevatecher.github.io/standard-methods-apis-omics/Section_7_1/)

## [7.2. CRISPR](https://maevatecher.github.io/standard-methods-apis-omics/Section_7_2/)

### [7.2.1. Considerations](https://maevatecher.github.io/standard-methods-apis-omics/Section_7_2/#721-considerations)

### [7.2.2. Materials](https://maevatecher.github.io/standard-methods-apis-omics/Section_7_2/#722-materials)

### [7.2.3. Methods for CRISPR/Cas9 gene editing of embryos](https://maevatecher.github.io/standard-methods-apis-omics/Section_7_2/#723-methods-for-crisprcas9-gene-editing-of-embryos)

#### [7.2.3.1. Generating Cas9 protein](https://maevatecher.github.io/standard-methods-apis-omics/Section_7_2/#7231-generating-cas9-protein)

#### [7.2.3.2. Generating sgRNA](https://maevatecher.github.io/standard-methods-apis-omics/Section_7_2/#7232-generating-sgrna)

#### [7.2.3.3. Ribonucleoprotein assembly](https://maevatecher.github.io/standard-methods-apis-omics/Section_7_2/#7233-ribonucleoprotein-assembly)

#### [7.2.3.4. Egg collection and microinjection](https://maevatecher.github.io/standard-methods-apis-omics/Section_7_2/#7234-egg-collection-and-microinjection)

## [7.3. RNA interference](https://maevatecher.github.io/standard-methods-apis-omics/Section_7_3/)

### [7.3.1. RNAi considerations](https://maevatecher.github.io/standard-methods-apis-omics/Section_7_3/#731-rnai-considerations)

### [7.3.2. Methods for nanoparticle-mediated RNAi](https://maevatecher.github.io/standard-methods-apis-omics/Section_7_3/#732-methods-for-nanoparticle-mediated-rnai)

#### [7.3.2.1. Materials](https://maevatecher.github.io/standard-methods-apis-omics/Section_7_3/#7321-materials)

#### [7.3.2.2. Procedure](https://maevatecher.github.io/standard-methods-apis-omics/Section_7_3/#7322-procedure)

## [7.4. Xenobiotic treatment](https://maevatecher.github.io/standard-methods-apis-omics/Section_7_4/)

### [7.4.1 Xenobiotic treatment considerations](https://maevatecher.github.io/standard-methods-apis-omics/Section_7_4/#741-xenobiotic-treatment-considerations)

### [7.4.2. Materials](https://maevatecher.github.io/standard-methods-apis-omics/Section_7_4/#742-materials)

### [7.4.3 Procedure](https://maevatecher.github.io/standard-methods-apis-omics/Section_7_4/#743-procedure)

#### [7.4.3.1 Thorax application](https://maevatecher.github.io/standard-methods-apis-omics/Section_7_4/#7431-thorax-application)

#### [7.4.3.2. Injection](https://maevatecher.github.io/standard-methods-apis-omics/Section_7_4/#7432-injection)

#### [7.4.3.3. Feeding individual bees](https://maevatecher.github.io/standard-methods-apis-omics/Section_7_4/#7433-feeding-individual-bees)

#### [7.4.3.4. Flight cage feeding](https://maevatecher.github.io/standard-methods-apis-omics/Section_7_4/#7434-flight-cage-feeding)

## [7.5. Applications and limitations](https://maevatecher.github.io/standard-methods-apis-omics/Section_7_5/)

# [8. Proteomics](https://maevatecher.github.io/standard-methods-apis-omics/Section_8_1/)

## [8.1. Introduction](https://maevatecher.github.io/standard-methods-apis-omics/Section_8_1/)

## [8.2. Standard methods for shot-gun proteomics sample preparation](https://maevatecher.github.io/standard-methods-apis-omics/Section_8_2/)

### [8.2.1. Considerations](https://maevatecher.github.io/standard-methods-apis-omics/Section_8_2/#821-considerations)

#### [8.2.1.1. General](https://maevatecher.github.io/standard-methods-apis-omics/Section_8_2/#8211-general)

#### [8.2.1.2. Sample handling](https://maevatecher.github.io/standard-methods-apis-omics/Section_8_2/#8212-sample-handling)

#### [8.2.1.3. Reagent handling](https://maevatecher.github.io/standard-methods-apis-omics/Section_8_2/#8213-reagent-handling)

### [8.2.2. Materials](https://maevatecher.github.io/standard-methods-apis-omics/Section_8_2/#822-materials)

### [8.2.3. Proteomics methods](https://maevatecher.github.io/standard-methods-apis-omics/Section_8_2/#823-proteomics-methods)

#### [8.2.3.1. Lysis and precipitation](https://maevatecher.github.io/standard-methods-apis-omics/Section_8_2/#8231-lysis-and-precipitation)

#### [8.2.3.2. Solubilization and digestion](https://maevatecher.github.io/standard-methods-apis-omics/Section_8_2/#8232-solubilization-and-digestion)

#### [8.2.3.3. Peptide desalting and resuspension](https://maevatecher.github.io/standard-methods-apis-omics/Section_8_2/#8233-peptide-desalting-and-resuspension)

## [8.3. Liquid chromatography and mass spectrometry](https://maevatecher.github.io/standard-methods-apis-omics/Section_8_3/)

## [8.4. Proteomics data processing](https://maevatecher.github.io/standard-methods-apis-omics/Section_8_4/)

### [8.4.1. Software recommendations](https://maevatecher.github.io/standard-methods-apis-omics/Section_8_4/#841-software-recommendations)

#### [8.4.1.1. MaxQuant and DIA-NN search parameters](https://maevatecher.github.io/standard-methods-apis-omics/Section_8_4/#8411-maxquant-and-dia-nn-search-parameters)

#### [8.4.1.2. Choosing an appropriate protein database](https://maevatecher.github.io/standard-methods-apis-omics/Section_8_4/#8412-choosing-an-appropriate-protein-database)

#### [8.4.1.3 Statistical analysis](https://maevatecher.github.io/standard-methods-apis-omics/Section_8_4/#8413-statistical-analysis)

## [8.5. Applications and limitations](https://maevatecher.github.io/standard-methods-apis-omics/Section_8_5/)

# [9. Metabolomics](https://maevatecher.github.io/standard-methods-apis-omics/Section_9_1/)

## [9.1. Introduction](https://maevatecher.github.io/standard-methods-apis-omics/Section_9_1/)

## [9.2. Sample preparation for metabolomics](https://maevatecher.github.io/standard-methods-apis-omics/Section_9_2/)

### [9.2.1. Considerations](https://maevatecher.github.io/standard-methods-apis-omics/Section_9_2/#921-considerations)

#### [9.2.1.1. General](https://maevatecher.github.io/standard-methods-apis-omics/Section_9_2/#9211-general)

#### [9.2.1.2. Sample handling](https://maevatecher.github.io/standard-methods-apis-omics/Section_9_2/#9212-sample-handling)

#### [9.2.1.3. Reagent handling](https://maevatecher.github.io/standard-methods-apis-omics/Section_9_2/#9213-reagent-handling)

#### [9.2.2. Materials](https://maevatecher.github.io/standard-methods-apis-omics/Section_9_2/#922-materials)

### [9.2.3. Metabolomics methods](https://maevatecher.github.io/standard-methods-apis-omics/Section_9_2/#923-metabolomics-methods)

#### [9.2.3.1. Sample homogenization](https://maevatecher.github.io/standard-methods-apis-omics/Section_9_2/#9231-sample-homogenization)

#### [9.2.3.2. Extraction](https://maevatecher.github.io/standard-methods-apis-omics/Section_9_2/#9232-extraction)

## [9.3. Chromatography and mass spectrometry](https://maevatecher.github.io/standard-methods-apis-omics/Section_9_3/)

## [9.4. Metabolomics data processing](https://maevatecher.github.io/standard-methods-apis-omics/Section_9_4/)

## [9.5. Metabolomics applications and limitations](https://maevatecher.github.io/standard-methods-apis-omics/Section_9_5/)

# [10. Microbiome analysis](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_1/)

## [10.1. Introduction](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_1/)

## [10.2. Sampling and DNA extraction](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/)

### [10.2.1. Considerations](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/#1021-considerations)

#### [10.2.1.1. General](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/#10211-general)

#### [10.2.1.2. Tissue sample handling](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/#10212-tissue-sample-handling)

#### [10.2.1.3. Hive material sample handling](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/#10213-hive-material-sample-handling)

### [10.2.2. Protocol for tissue samples](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/#1022-protocol-for-tissue-samples)

#### [10.2.2.1. Materials](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/#10221-materials)

#### [10.2.2.2. Dissection methods](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/#10222-dissection-methods)

#### [10.2.2.3. DNA extraction methods](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/#10223-dna-extraction-methods)

### [10.2.3. Protocol for sampling hive materials](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/#1023-protocol-for-sampling-hive-materials)

#### [10.2.3.1. Materials](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/#10231-materials)

#### [10.2.3.2. Methods for bee bread sampling and DNA extraction](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/#10232-methods-for-bee-bread-sampling-and-dna-extraction)

#### [10.2.3.3. Methods for hive entrance sampling and DNA extraction](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_2/#10233-methods-for-hive-entrance-sampling-and-dna-extraction)

## [10.3. Amplicon sequencing](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_3/)

### [10.3.1. Considerations](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_3/#1031-considerations)

### [10.3.2. Materials for amplicon sequencing](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_3/#1032-materials-for-amplicon-sequencing)

### [10.3.3. Methods for amplicon sequencing](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_3/#1033-methods-for-amplicon-sequencing)

## [10.4. Microbiome data analysis](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_4/)

### [10.4.1. Recommended software](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_4/#1041-recommended-software)

### [10.4.2. Guidance on the data analysis methods: An example with QIIME 2](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_4/#1042-guidance-on-the-data-analysis-methods-an-example-with-qiime-2)

#### [10.4.2.1. Importing data](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_4/#10421-importing-data)

#### [10.4.2.2. Non-biological sequence removal](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_4/#10422-non-biological-sequence-removal)

#### [10.4.2.3. Sequence quality control (denoising)](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_4/#10423-sequence-quality-control-denoising)

#### [10.4.2.4. Removing biological contamination](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_4/#10424-removing-biological-contamination)

## [10.5. Applications and limitations](https://maevatecher.github.io/standard-methods-apis-omics/Section_10_5/)

# [11. Data management and open access sharing](https://maevatecher.github.io/standard-methods-apis-omics/Section_11_1/)

## [11.1. Metadata standardization](https://maevatecher.github.io/standard-methods-apis-omics/Section_11_1/)

### [11.1.1. Common problems with Apis-related BioProjects](https://maevatecher.github.io/standard-methods-apis-omics/Section_11_1/#1111-common-problems-with-apis-related-bioprojects)

### [11.1.2. Common problems with Apis-related BioSamples](https://maevatecher.github.io/standard-methods-apis-omics/Section_11_1/#1112-common-problems-with-apis-related-biosamples)

## [11.2. Sharing pipelines and scripts](https://maevatecher.github.io/standard-methods-apis-omics/Section_11_2/)

# [12. The future of Apis omics: Biological integration](https://maevatecher.github.io/standard-methods-apis-omics/Section_12_1/)
