# 6. Transcriptomics

## 6.1. Introduction

Transcriptomics is the study of expressed transcripts, differential splicing patterns of messenger RNA, or chemical modifications on RNAs. ***What mechanisms program or influence genes expression, regulation, interaction?*** These questions can be answered holistically by sequencing the transcriptome and aligning it against an annotated reference genome.

Before NGS became accessible and affordable, northern blot techniques were most often used to detect and quantify mRNA transcripts. Quantitative reverse transcription-polymerase chain reaction (qRT-PCR) has also been used to answer similar questions. However, each of these techniques is practically limited to a small number of targeted genes. Transcriptomic techniques can identify nearly every transcript in an organism or tissue, and the relative quantities of these transcripts can be compared, providing valuable insights into the gene regulatory networks involved in a phenotype.

As honey bees were among the first insect genomes sequenced, the study of gene expression and regulations in *Apis* had the tremendous advantage of having an annotated reference genome as an anchor to map transcripts. At the time of publication of Evans *et al.* (2013), microarrays were the most commonly used high-throughput method of evaluating transcript abundances. Since then, RNA-seq has become a mainstay, with multiple sequencing platforms available, and even single-cell transcriptomics is possible. Here, we address these new techniques and how they can be applied to honey bee samples.
